% JAIDS author guidelines: http://edmgr.ovid.com/jaids/accounts/ifauth.htm
%DIF LATEXDIFF DIFFERENCE FILE
%DIF DEL old/main.tex   Thu Dec  3 16:32:10 2020
%DIF ADD new/main.tex   Tue Dec  8 16:24:38 2020

\documentclass[a4paper,11pt]{article}
\usepackage{geometry}
\geometry{margin=1in}

\usepackage[style= nature, autocite = superscript, sorting=none, firstinits=true, terseinits=true, minnames=1, minbibnames=3,maxbibnames=3, url=false, doi=true,isbn=false, mincitenames=1, maxcitenames=3, maxnames=3]{biblatex}
\DeclareNameAlias{author}{last-first}
\renewcommand*{\revsdnamepunct}{}
\addbibresource{proact.bib}
\addbibresource{smreferences.bib}

\newcommand{\authorcite}[1]{\citeauthor{#1}\supercite{#1}}

\usepackage{lineno}


\usepackage{setspace}
\onehalfspacing

\usepackage{url}

\usepackage{indentfirst}

\usepackage{amsmath}

\usepackage{xcolor}
%\newcommand{\CHECK}[1]{ {\color{blue} #1} }
\newcommand{\TODO}[1]{{\color{red} #1} }

\newcommand{\beginsupplement}{%
        \setcounter{table}{0}
        \renewcommand{\thetable}{S\arabic{table}}%
        \setcounter{figure}{0}
        \renewcommand{\thefigure}{S\arabic{figure}}%
        \setcounter{section}{0}
        \renewcommand{\thesection}{S\arabic{section}}%
        \clearpage%\onecolumn
        \begin{center}
        ~\\{Supplementary Materials}\\~\\
        \end{center}
}

\usepackage{xspace}
\newcommand{\bigO}{\mathcal{O}}
%\newcommand{\PLH}{PLWH\xspace}
\newcommand{\PLWH}{sample\xspace}

\usepackage{booktabs}
\usepackage{arydshln}

\usepackage{xfrac}

\newcommand{\D}{D}
\newcommand{\ta}[1][A]{t'}
\newcommand{\tb}{t'}
%DIF PREAMBLE EXTENSION ADDED BY LATEXDIFF
%DIF UNDERLINE PREAMBLE %DIF PREAMBLE
\RequirePackage[normalem]{ulem} %DIF PREAMBLE
\RequirePackage{color}\definecolor{RED}{rgb}{1,0,0}\definecolor{BLUE}{rgb}{0,0,1} %DIF PREAMBLE
\providecommand{\DIFadd}[1]{{\protect\color{blue}\uwave{#1}}} %DIF PREAMBLE
\providecommand{\DIFdel}[1]{{\protect\color{red}\sout{#1}}}                      %DIF PREAMBLE
%DIF SAFE PREAMBLE %DIF PREAMBLE
\providecommand{\DIFaddbegin}{} %DIF PREAMBLE
\providecommand{\DIFaddend}{} %DIF PREAMBLE
\providecommand{\DIFdelbegin}{} %DIF PREAMBLE
\providecommand{\DIFdelend}{} %DIF PREAMBLE
%DIF FLOATSAFE PREAMBLE %DIF PREAMBLE
\providecommand{\DIFaddFL}[1]{\DIFadd{#1}} %DIF PREAMBLE
\providecommand{\DIFdelFL}[1]{\DIFdel{#1}} %DIF PREAMBLE
\providecommand{\DIFaddbeginFL}{} %DIF PREAMBLE
\providecommand{\DIFaddendFL}{} %DIF PREAMBLE
\providecommand{\DIFdelbeginFL}{} %DIF PREAMBLE
\providecommand{\DIFdelendFL}{} %DIF PREAMBLE
%DIF END PREAMBLE EXTENSION ADDED BY LATEXDIFF

\begin{document}

\title{\textbf{Title Page}\\~\\~\\HIV Care Prioritization using Phylogenetic Branch Length}

\author{Niema Moshiri, PhD$^{1}$, Davey M. Smith, MD$^{2}$, and
Siavash Mirarab, PhD$^{3}$\\[4pt]}
\markboth{Moshiri, Smith, Mirarab}{ProACT}
\date{}

\maketitle

\textit{$^{1}$~Department of Computer Science and Engineering, University of California, San Diego, La Jolla, 92093, USA}
\\
\textit{$^{2}$~Department of Medicine, University of California, San Diego, La Jolla, 92093, USA}
\\
\textit{$^{3}$~Department of Electrical and Computer Engineering, University of California, San Diego, La Jolla, 92093, USA}
\\[2pt]

\textit{Correspondance: Siavash Mirarab, PhD, Department of Electrical and Computer Engineering, University of California, San Diego, 9500 Gilman Drive, Mail Code 0407, La Jolla, USA, 92093-0407, 858-822-6245, smirarab@ucsd.edu}\\~\\~\\


{\bf Funding: }
This work was supported by the National Institutes of Health (5P30AI027767-28, AI100665, AI106039, and MH100974) and a developmental grant from the University of California, San Diego Center for AIDS Research (P30 AI036214), supported by the National Institutes of Health.

~\\~\\

\textbf{Running Head:} ProACT

\pagebreak

\begin{abstract}{
\textbf{Background} -- The structure of the HIV transmission networks can be dictated by just a few individuals. 
Public health intervention, such as ensuring people living with HIV adhere to antiretroviral therapy and remain virally-suppressed, can help control the spread of the virus. However, such intervention requires utilizing  limited public health resource allocations. Determining which individuals are most at risk of transmitting HIV could allow public health officials to focus their limited resources on these individuals. 

\textbf{Setting} -- 
Molecular epidemiology can help prioritize people living with HIV by patterns of transmission inferred from their sampled viral sequences.
Such prioritization has been previously suggested and performed by monitoring cluster growth. 
In this paper, we introduce ProACT (\textbf{Pr}i\textbf{o}ritization using \textbf{A}n\textbf{C}es\textbf{T}ral edge lengths), a phylogenetic approach for prioritizing individuals living with HIV.

\textbf{Methods} -- 
ProACT starts from a phylogeny inferred from sequence data and orders individuals according to their terminal branch length, breaking ties using ancestral branch lengths. 
We evaluated ProACT on a real dataset of 926 HIV-1 subtype B \textit{pol}  data obtained in San Diego between 2005 and 2014, and a simulation datasets modelling the same epidemic. 
Prioritization methods are compared by their ability to predict individuals who transmit most \textit{after} the prioritization.
%real dataset of 926 HIV-1 subtype B \textit{pol} sequences was utilized.
%Individuals were then prioritized using ProACT and by monitoring cluster growth.

\textbf{Results} -- Across all simulation conditions and most real data sampling conditions, ProACT outperformed monitoring cluster growth for multiple metrics of prioritization efficacy.

\textbf{Conclusion} -- The simple strategy used by ProACT improves the effectiveness of prioritization compared to state-of-the-art methods that rely on monitoring the growth of transmission clusters defined based on genetic distance.}
%\bf{Keywords:} HIV, epidemiology, phylogenetics
\end{abstract}

\pagebreak

\linenumbers


\section{Introduction}

The transmission of Human Immunodeficiency Virus (HIV) resembles scale-free networks,\supercite{Wertheim2014}
\DIFdelbegin \DIFdel{in which the majority of the structure of the network is dictated by just a few individuals,
a phenomenon likely resulting from }\DIFdelend %DIF > in which the majority of the structure of the network is dictated by just a few individuals,
 \DIFaddbegin \DIFadd{likely due to }\DIFaddend the scale-free properties of sexual contacts and injection drug use\DIFdelbegin \DIFdel{along which HIV is transmitted}\DIFdelend .\supercite{Little2014,Schneeberger2004}
As a result, public health intervention may be more effective when targeted at people living with HIV  who are more likely to grow the transmission network.
However, the best method to target individuals for specific interventions remains an open question\DIFdelbegin \DIFdel{, and
the best strategy will likely depend on the specific intervention planned.
}\DIFdelend \DIFaddbegin \DIFadd{.
%DIF > , and the best strategy will likely depend on the specific intervention planned.
}\DIFaddend 

\DIFdelbegin \DIFdel{One form of intervention aiming to reduce future transmissionsis continued treatment}\DIFdelend \DIFaddbegin \DIFadd{Continued treatment is one way of reducing  transmissions}\DIFaddend . 
Antiretroviral therapy (ART) \DIFdelbegin \DIFdel{is an effective treatment of HIV that }\DIFdelend \DIFaddbegin \DIFadd{effectively }\DIFaddend suppresses the HIV virus in the majority of cases, stops the progression of the disease, and prevents onward transmission\DIFdelbegin \DIFdel{to an uninfected sexual partner}\DIFdelend , provided the individual continuously adheres to the treatment.\supercite{Cohen2011}
%In addition to reducing risk of transmission at the molecular level, adherence to ART is associated with %may yield
%a reduction of risky behavior as well \supercite{Bunnell2006}.
In  many (though not all) health care systems, ART is made available routinely  to newly diagnosed patients, but \DIFdelbegin \DIFdel{several opportunities for further intervention remain available. 
Most importantly, not }\DIFdelend \DIFaddbegin \DIFadd{further care is needed. 
Not }\DIFaddend every diagnosed person initiates ART and not all cases of ART initiation lead to a sustained suppression  \DIFdelbegin \DIFdel{of the virus }\DIFdelend through time. 
Individuals who start ART but fail to sustain it or who are otherwise unsuppressed can still infect others. 
Thus, a possible intervention is to \DIFdelbegin \DIFdel{use public health resources to }\DIFdelend help diagnosed individuals to stay on ART and to remain continually suppressed.\supercite{Poon2016}
Such interventions require allocation of clinical staff who would follow up with patients to provide them further assistance in  \DIFdelbegin \DIFdel{adherence }\DIFdelend sustenance of ART.
The health system can also provide increased \DIFdelbegin \DIFdel{testing to these }\DIFdelend \DIFaddbegin \DIFadd{viral load testing to adhering }\DIFaddend individuals to ensure suppression.  

\DIFdelbegin \DIFdel{A second family of interventions involves targeting HIV negative and undiagnosed HIV positive }\DIFdelend \DIFaddbegin \DIFadd{Another form of intervention targets undiagnosed and HIV negative  }\DIFaddend individuals.
The health system can \DIFdelbegin \DIFdel{potentially }\DIFdelend use partner tracing to identify the sexual partners of \DIFdelbegin \DIFdel{high-priority diagnosed individuals (}\DIFdelend \DIFaddbegin \DIFadd{diagnosed individuals }\DIFaddend as best as possible\DIFdelbegin \DIFdel{)}\DIFdelend , test these \DIFdelbegin \DIFdel{high risk }\DIFdelend individuals, and offer them either treatment \DIFdelbegin \DIFdel{(for positives ) or  prevention through PrEP (for negatives)}\DIFdelend \DIFaddbegin \DIFadd{for positives or  PrEP prevention for negatives}\DIFaddend .\supercite{Gotz2014} 
\DIFdelbegin \DIFdel{Finally, if the  priority status of individuals shows any association with specific geographical or demographic groups (beyond known associations), the public health system can design strategies for further outreach, testing, and PrEP administration for the impacted groups.}\DIFdelend \DIFaddbegin \DIFadd{Although contact tracing may have ethical concerns, it is a common and appropriate tool for  controlling infectious diseases (e.g. HIV, Tb, SARS-CoV-2) when delivered by trained public health staff. }\supercite{Guttman2020} 
\DIFadd{Contact tracing for HIV is resource-intensive and is often viewed as insufficiently  used.}\supercite{Katz2010} 
\DIFadd{Community-level reductions in viral loads are associated with decreased HIV incidence rates}\supercite{Das2010} \DIFadd{(though empirical data on effectiveness of contact tracing are scant.}\supercite{Hogben2007}\DIFadd{)
}\DIFaddend 

\DIFdelbegin \DIFdel{All three types of intervention   are }\DIFdelend \DIFaddbegin \DIFadd{These interventions are }\DIFaddend costly and cannot be undertaken for every known individual.
If diagnosed people at risk of not being suppressed \DIFdelbegin \DIFdel{could }\DIFdelend \DIFaddbegin \DIFadd{can }\DIFaddend be predicted accurately, the public health system \DIFdelbegin \DIFdel{could focus their limited resources on }\DIFdelend \DIFaddbegin \DIFadd{can prioritize followup efforts to }\DIFaddend these individuals.
\DIFdelbegin \DIFdel{Thus, a natural question surfaces: which individuals are most at-risk of transmitting HIV? %DIF < the virus?
}\DIFdelend \DIFaddbegin \DIFadd{For tracing, care provided to positive contacts and PrEP given to high-risk negative contacts can  decrease HIV spread within the community }\textit{\DIFadd{only if}} \DIFadd{enough unsuppressed positives and high-risk contacts are found; thus, individuals with many such contacts should be prioritized.
Moreover, if these high-priority individuals show  associations with geographical or demographic groups (beyond what is known), the public health system can design strategies for further outreach, testing, and PrEP administration for the impacted groups.
The goal of prioritization is to increase the number of prevented transmissions for a fixed budget by increasing the chance of finding the right targets for followups, tracing, and testing.
%DIF > The question is: which individuals are most at-risk of transmitting HIV? %the virus?
}\DIFaddend However, predicting tendency for future transmissions is difficult, especially if we \DIFdelbegin \DIFdel{want }\DIFdelend \DIFaddbegin \DIFadd{aim }\DIFaddend to be more specific than  \DIFdelbegin \DIFdel{simply relying on known broad demographical characteristics. 
%DIF < and can also be problematic if undertaken primarily based on demographic or behavioral traits.
}\DIFdelend \DIFaddbegin \DIFadd{broad demographic characteristics. 
}\DIFaddend 

\DIFdelbegin \DIFdel{Molecular epidemiology suggest an alternative method:
prioritizing   for intervention  }\DIFdelend %DIF > and can also be problematic if undertaken primarily based on demographic or behavioral traits.
\DIFaddbegin \DIFadd{Molecular epidemiology suggests a solution:
prioritizing   }\DIFaddend based on patterns of transmission inferred from HIV sequence data.\supercite{Bbosa2019,Villandre2019,Oster2018,Ragonnet-Cronin2019,Wertheim2018,Wertheim2011,Wertheim2014,Smith2009}
The inference of transmission networks using phylogenetic or distance-based methods has been the subject of much research.\supercite{Leitner2018,Pond2018,Ragonnet-Cronin2013,Prosperi2011}
However, in this work, instead of being concerned with inferring exact patterns of transmissions, we ask the following question:
given molecular data from a set of \textit{sequenced} HIV positive individuals (``\PLWH{s}'' for short), %presumably all with access to ART, 
who should be prioritized for further intervention? 

Prioritizing care based on molecular epidemiology has been studied recently. 
\authorcite{Wertheim2018} present a method for prioritizing \PLWH{s} based on performing transmission clustering (i.e., grouping individuals with low viral genetic distance into \textit{transmission clusters}) and ordering clusters  by growth rate.
On a large dataset from New York, they show that the approach is able to predict individuals who have relatively larger numbers of transmission links in the near future.
\authorcite{Moshiri2018} have studied the same question in simulations and have shown that monitoring cluster growth can be used for predicting future transmissions substantially better than a random guess, whether clusters are defined using genetic distances or using phylogenetic methods. 
\DIFdelbegin \DIFdel{Most recently, }\DIFdelend \authorcite{Balaban2019}  showed in simulations that using \DIFdelbegin \DIFdel{a cluster-monitoring }\DIFdelend \DIFaddbegin \DIFadd{an }\DIFaddend approach similar to that of \authorcite{Wertheim2018}  but defining clusters using \DIFdelbegin \DIFdel{a min-cut optimization problem }\DIFdelend \DIFaddbegin \DIFadd{phylogenetic trees }\DIFaddend gives a small but consistent improvement over \DIFdelbegin \DIFdel{defining clusters using genetic }\DIFdelend \DIFaddbegin \DIFadd{clusters defined using  }\DIFaddend distances. 
\authorcite{McLaughlin2019} developed a predictive model in which they combined viral lineage-level diversification rates inferred from phylogenetic trees with sociodemographic and clinical data, and after aggregating by patients' area of residence, they were able to predict the distribution of new HIV cases in British Columbia, Canada between 2008 and 2018 with higher predictive power than a phylogenetically uninformed model.

%The viral phylogeny is dependent upon the transmission history of the virus, and as a result, properties of the transmission history can be inferred from the viral phylogeny \supercite{Leitner2018}.
In this paper, we introduce a new method for ordering \PLWH{s} based  on their phylogenetic relationships. Instead of relying on clustering individuals and then ordering clusters based on their growth, we seek to order individuals without clustering, without relying on parametric models, and without incorporating demographic information about patients. 
Instead, we seek to simply \DIFdelbegin \DIFdel{exploits }\DIFdelend \DIFaddbegin \DIFadd{exploit }\DIFaddend patterns in the phylogeny, and in particular, in branch lengths. 
%and other attributes related to the recency of their infection.


\section{Methods}
ProACT (\textbf{Pr}i\textbf{o}ritization using \textbf{A}n\textbf{C}es\textbf{T}ral edge lengths) takes as input the inferred phylogenetic \DIFdelbegin \DIFdel{relationships between }\DIFdelend \DIFaddbegin \DIFadd{tree of }\DIFaddend sampled HIV viruses\DIFdelbegin \DIFdel{(e.g. from the }\textit{\DIFdel{pol}} %DIFAUXCMD
\DIFdel{region), }\DIFdelend \DIFaddbegin \DIFadd{, %DIF > (e.g. from the \textit{pol} region), 
}\DIFaddend rooted using an outgroup or clock-based methods (e.g.\DIFdelbegin \DIFdel{midpoint or }\DIFdelend \DIFaddbegin \DIFadd{, }\DIFaddend MinVar-root\supercite{Mai2017}). 
ProACT simply orders \PLWH{s} in \DIFaddbegin \DIFadd{ascending }\DIFaddend order of incident branch length of their associated virus,  \DIFdelbegin \DIFdel{and it breaks }\DIFdelend \DIFaddbegin \DIFadd{breaking }\DIFaddend ties based on \DIFaddbegin \DIFadd{the }\DIFaddend incident branch lengths of parent nodes, then those of grandparent nodes, etc.\DIFdelbegin \DIFdel{We first motivate the approach and then present a formal definition of the method.
}\DIFdelend \DIFaddbegin \DIFadd{, and ultimately by sampling times if available (see Supplementary Figure }{\DIFadd{S1}} \DIFadd{for a formal description).
ProACT is very fast (scaling quasilinearly), and scalable methods exist  for  inferring~}\supercite{Price2010,Nguyen2015} \DIFadd{and rooting~}\supercite{Mai2017} \DIFadd{very large trees.
%DIF > We next motivate the approach.
}\DIFaddend 


\DIFdelbegin \subsection{\DIFdel{Motivating the ProACT Approach}}
%DIFAUXCMD
\addtocounter{subsection}{-1}%DIFAUXCMD
\DIFdelend %DIF > \subsection{Motivating the ProACT Approach}
ProACT is motivated and tested in a context similar to those  health care systems that enjoy enough resources to \textit{offer} ART to  all (or \DIFdelbegin \DIFdel{at least }\DIFdelend most) diagnosed individuals.
Thus, each \PLWH can be assumed to be given ART at a time close to when their HIV is sequenced, but they may fail to be suppressed for the remainder of their life.
These conditions describe the current common practice of care in many places and is increasingly adopted elsewhere. %many high-income and (increasingly) low-to-middle income countries. 



\DIFdelbegin \DIFdel{We }\DIFdelend \DIFaddbegin \DIFadd{To motivate ProACT, we }\DIFaddend start with the observation that, in \DIFdelbegin \DIFdel{simulations }\DIFdelend \DIFaddbegin \DIFadd{a ten-year epidemic simulation }\DIFaddend (described in detail below),
when a phylogeny is inferred from sequences obtained at a given time point in an epidemic,
the more a node transmits, the shorter its incident branch length tends to be
%the individuals who are most at-risk of transmitting the virus in the near future have short incident branch lengths 
(Figs.~1d--e and~S2).
%The more a node transmits, the shorter its length tends to be (Fig.~1d).
Using the Kendall's tau-b test,\supercite{Kendall1938} \DIFdelbegin \DIFdel{in a ten-year epidemic simulation, }\DIFdelend we found a statistically significant anticorrelation between the incident branch lengths of individuals sampled within the first 9 years of the epidemic and the number of individuals they infected \DIFdelbegin \DIFdel{over the final year of the epidemic. This held for }\DIFdelend \DIFaddbegin \DIFadd{during year 10, regardless of whether we use  }\DIFaddend true ($\tau_B=-0.0431$, $p\ll 10^{-10}$) \DIFdelbegin \DIFdel{and }\DIFdelend \DIFaddbegin \DIFadd{or }\DIFaddend inferred ($\tau_B=-0.0354$, $p\ll 10^{-10}$) \DIFdelbegin \DIFdel{phylogenetic trees}\DIFdelend \DIFaddbegin \DIFadd{phylogenies}\DIFaddend .
%In fact, as we will show, there is a statistically significant correlation between terminal branch length and propensity to transmit in many situations.
Though not obvious, this observation can be explained by the constraints placed upon the viral phylogeny by the transmission history (Fig.~1a--c).

\begin{figure}[!tp]
\centering
%\includegraphics[width=0.995\textwidth]{figs/diagram.pdf}\\
%\includegraphics[width=0.425\textwidth]{figs/avg_edgelength_vs_time_ftmv.pdf}%
%\includegraphics[width=0.57\textwidth]{figs/edgelength_vs_efficacy.pdf}\\\vspace{-5pt}
%~\hfill (d)\hfill~~~~~\hfill
%(e)\hfill~\\~\\
\includegraphics[width=\textwidth]{figs/Fig1.pdf}
\caption{\DIFdelbeginFL \DIFdelFL{The effect of new transmissions on incident branch lengths. }\DIFdelendFL  (a) Individual $A$ transmits to individual $B$ and $C$ at times at  $t_1$ and $t_2$\DIFdelbeginFL \DIFdelFL{, respectively}\DIFdelendFL . (b) Viral samples are obtained \DIFdelbeginFL \DIFdelFL{from individuals $A$, $B$, and $C$ }\DIFdelendFL %DIF > from individuals $A$, $B$, and $C$ 
at times $t_A$, $t_B$, and $t_C$. The viral phylogeny \DIFdelbeginFL \DIFdelFL{of samples }\DIFdelendFL is constrained by each transmission event's bottleneck\DIFdelbeginFL \DIFdelFL{, and the }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{.
The }\DIFaddendFL most likely phylogeny matches the transmission history (Left), but \DIFdelbeginFL \DIFdelFL{in the }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{with deeper coalescences (}\DIFaddendFL less likely\DIFdelbeginFL \DIFdelFL{deeper coalescence}\DIFdelendFL \DIFaddbeginFL \DIFaddFL{)}\DIFaddendFL , it may not match (Right). (c) Moving from \DIFdelbeginFL \DIFdelFL{the phylogeny observed at time }\DIFdelendFL %DIF > the phylogeny observed at time 
$t_B$ to
\DIFdelbeginFL \DIFdelFL{the phylogeny at time }\DIFdelendFL %DIF > the phylogeny at time 
$t_C$, the branch length incident to individual A  shortens upon the addition of individual C  in the likely event that the coalescence of the lineage from $C$ with the lineage from $A$ is more recent than its coalescence with the lineage from $B$ (Left), or 
\DIFdelbeginFL \DIFdelFL{the branch length incident to individual A }\DIFdelendFL %DIF > the branch length incident to individual A 
\DIFaddbeginFL \DIFaddFL{it }\DIFaddendFL remains constant in the event of a less likely deeper coalescence (Right). Regardless, the length of \DIFdelbeginFL \DIFdelFL{the }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{this }\DIFaddendFL branch  \DIFdelbeginFL \DIFdelFL{incident to individual A }\DIFdelendFL never increases.
In simulation, we can observe this trend: as time progresses, the incident branch \DIFdelbeginFL \DIFdelFL{length of each individual tends }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{lengths %DIF >  of each individual 
tend }\DIFaddendFL to decrease, both in true (Fig.~\DIFdelbeginFL \DIFdelFL{S1}\DIFdelendFL \DIFaddbeginFL \DIFaddFL{S2}\DIFaddendFL ) and inferred (d) phylogenies, and as the number of transmissions from a given individual increases, \DIFdelbeginFL \DIFdelFL{the distribution of }\DIFdelendFL incident edge \DIFdelbeginFL \DIFdelFL{length tends }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{lengths tend }\DIFaddendFL to decrease. Patterns are similar \DIFdelbeginFL \DIFdelFL{if we use }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{with }\DIFaddendFL the true simulated tree \DIFaddbeginFL \DIFaddFL{(``True'') }\DIFaddendFL or the tree inferred from sequences \DIFdelbeginFL \DIFdelFL{, labeled }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{(}\DIFaddendFL ``\DIFdelbeginFL \DIFdelFL{True'' and ``}\DIFdelendFL Est.\DIFdelbeginFL \DIFdelFL{,}\DIFdelendFL ''\DIFdelbeginFL \DIFdelFL{respectively }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{) }\DIFaddendFL (e).}
\label{fig:diagram}
\end{figure}


In \DIFdelbegin \DIFdel{the context of HIV epidemiology in }\DIFdelend many places, \DIFaddbegin \DIFadd{HIV
}\DIFaddend \PLWH{s} are typically sequenced
upon beginning Antiretroviral Therapy (ART).
Let's assume for simplicity that every individual in the given dataset has at some point initiated ART\DIFdelbegin \DIFdel{,
meaning }\DIFdelend \DIFaddbegin \DIFadd{; then,
 }\DIFaddend future transmissions by individuals in the dataset \DIFdelbegin \DIFdel{must }\DIFdelend \DIFaddbegin \DIFadd{can }\DIFaddend happen only if the source stops ART or is otherwise unsuppressed. %for other reasons.
\DIFdelbegin \DIFdel{Given a viral phylogeny containing all known }%DIFDELCMD < \PLWH{s}%%%
\DIFdel{, if, }\DIFdelend \DIFaddbegin \DIFadd{If, }\DIFaddend in the future,
individual $u$  \DIFdelbegin \DIFdel{in the dataset }\DIFdelend transmits to individual $v$,
there are two possible scenarios regarding the placement of the leaf corresponding to $v$ in the existing  \DIFdelbegin \DIFdel{(true) }\DIFdelend phylogeny:
(1) $v$ is placed on the edge incident to $u$, so the edge incident to $u$ will shorten, or (2) $v$ is  \DIFdelbegin \DIFdel{not placed on the edge incident to $u$}\DIFdelend \DIFaddbegin \DIFadd{placed further up the tree}\DIFaddend , so the edge incident to $u$ will remain the same length.
Although Scenario 2 is possible,
Scenario 1 is more likely,\supercite{Romero-Severson2016}\DIFdelbegin \DIFdel{and note that the terminal branch lengths do not increase in either scenario.
}\DIFdelend \DIFaddbegin \DIFadd{.
%DIF > Note that the terminal branch lengths do not increase in either scenario. 
}\DIFaddend Thus, as time goes by, \DIFdelbegin \DIFdel{the }\DIFdelend \DIFaddbegin \DIFadd{a }\DIFaddend terminal branch can only shorten or stay fixed, and it will most often shorten because of  \DIFdelbegin \DIFdel{new transmissions by the }%DIFDELCMD < \PLWH %%%
\DIFdel{associated with that terminal }\DIFdelend \DIFaddbegin \DIFadd{transmissions from the individual represented by that }\DIFaddend branch.
This pattern, easily observed in simulations (Fig.~1d), leads to shorter branches for \PLWH{s} who have transmitted recently.
%Since short-terminal branch nodes are likely to have transmitted recently, they are also likely to transmit in the near future because they are likely not to be  suppressed. 
%This likely lack of suppression makes them a good candidate for intervention. 
\DIFdelbegin %DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{Note that }%DIFDELCMD < \PLWH{s} %%%
\DIFdel{who transmit are unsuppressed. }\DIFdelend %DIF > Note that \PLWH{s} who transmit are unsuppressed. 
The first time \DIFdelbegin \DIFdel{they }\DIFdelend \DIFaddbegin \DIFadd{a }\PLWH \DIFaddend infect others, \DIFdelbegin \DIFdel{their }\DIFdelend \DIFaddbegin \DIFadd{its }\DIFaddend terminal branch length \DIFdelbegin \DIFdel{is likely to decrease}\DIFdelend \DIFaddbegin \DIFadd{likely decreases}\DIFaddend , and further transmissions further decrease \DIFdelbegin \DIFdel{their terminal branch lengths }\DIFdelend \DIFaddbegin \DIFadd{the  length }\DIFaddend (Fig.~1d). 
Thus, one expects nodes with smaller incident branch length to be more likely to have transmitted since their sampling time.
%, and by extrapolation, in the near future.
%This pattern is what we observe in simulations (Fig.~S2).
Moreover, they are more likely not to be  suppressed compared to individuals on long branches. 
Since only individuals that are not suppressed transmit, a higher probability of a lack of suppression makes \DIFaddbegin \DIFadd{an }\DIFaddend individual a good candidate for intervention \DIFdelbegin \DIFdel{(}\DIFdelend to ensure supression\DIFdelbegin \DIFdel{)}\DIFdelend . 
%This simple observation, that nodes with short branches tend to transmit more often, is the central 

\DIFdelbegin \subsection{\DIFdel{Formal Description}}
%DIFAUXCMD
\addtocounter{subsection}{-1}%DIFAUXCMD
\DIFdel{ProACT takes as input a }%DIFDELCMD < {\em %%%
\DIFdel{rooted}%DIFDELCMD < } %%%
\DIFdel{phylogenetic tree $T$ of viral samples. %DIF < and a list of sample times.
Let $bl(u)$ denote the incident branch length of node $u$,
and assume the incident branch length of the root of $T$ is 0.
Let $a(u)$ denote the vector of ancestors of node $u$ (including $u$),
where $a(u)_1$ is $u$,
$a(u)_2$ is the parent of $u$,
$a(u)_3$ is the grandparent of $u$, etc.
Let $r(u)$ denote the length of the path from node $u$ to the root of $T$,
i.e., $r(u)=\sum_{v\in a(u)}{bl(v)}$.
ProACT sorts the leaves of $T$ in ascending order of $bl(a(u)_1)$,
with ties broken by $bl(a(u)_2)$, then by $bl(a(u)_3)$, etc.
Note that, for two leaves $u$ and $v$,
$|a(u)|$ may be less than $|a(v)|$,
in which case, for all $|a(u)|<i\le|a(v)|$,
$\frac{r(u)}{|a(u)|-1}$ (i.e., average branch length along the path from $u$ to the root of $T$) is compared with $bl(a(v)_i)$ instead.
If two nodes are equal in all comparisons, if the user provides sample times, the earlier sample time is given higher priority; otherwise, ties are broken arbitrarily.
%DIF < Given a rooted tree $T$, let $\D(u)$ denote the set of all leaf descendants of an internal node $u$ and 
%DIF < let $t(l)$ denote the sample time of leaf $l$.
%DIF < Let $\ta(u)=\sfrac{1}{\lvert \D(u)\rvert}\sum_{v\in \D(u)}{t(u)}$ denote the average sample time of leaf descendants below an internal $u$.
%DIF < Let $\bar{p}(l)$ be the path from $l$ to the root excluding $l$, and for each leaf $l$ let $\tb(l)=\min_{u\in \bar{p}(l)} \ta(u)$ be  the minimum average sample time assigned to any of it ancestor nodes (not including itself).
%DIF < ProACT orders leaves in the descending order (i.e., from the most recent to the oldest) of the tuple $(\tb(l),t(l))$.
%DIF < Note that $\ta(u)$ values can be computed in $\Theta(n)$ for all nodes $u$ in $T$ using a post-order traversal. 
Because sorting is needed, for a tree with $n$ leaves,
assuming branch lengths are fairly unique,
the ProACT algorithm runs in $\bigO(n\log n)$ time. Scalable methods exist  for the inferring~}%DIFDELCMD < \supercite{Price2010,Nguyen2015} %%%
\DIFdel{and rooting~}%DIFDELCMD < \supercite{Mai2017} %%%
\DIFdel{very large trees.
}\DIFdelend %DIF >  \subsection{Formal Description of ProACT}
%DIF >  %ProACT takes as input a {\em rooted} phylogenetic tree $T$ of viral samples. %and a list of sample times.
%DIF >  Let $bl(u)$ denote the incident branch length of node $u$  in the rooted input tree $T$.
%DIF >  %and assume the incident branch length of the root of $T$ is 0.
%DIF >  Let $a(u)$ denote the vector of ancestors of node $u$ (including $u$),
%DIF >  where $a(u)_1$ is $u$,
%DIF >  $a(u)_2$ is the parent of $u$,
%DIF >  $a(u)_3$ is the grandparent of $u$, etc.
%DIF >  Let $r(u)$ denote the length of the path from node $u$ to the root of $T$,
%DIF >  i.e., $r(u)=\sum_{v\in a(u)}{bl(v)}$.
%DIF >  ProACT sorts the leaves of $T$ in ascending order of $bl(a(u)_1)$,
%DIF >  with ties broken by $bl(a(u)_2)$, then by $bl(a(u)_3)$, etc.
%DIF >  Note that, for two leaves $u$ and $v$,
%DIF >  $|a(u)|$ may be less than $|a(v)|$,
%DIF >  in which case, for all $|a(u)|<i\le|a(v)|$,
%DIF >  $\frac{r(u)}{|a(u)|-1}$ (i.e., average branch length along the path from $u$ to the root of $T$) is compared with $bl(a(v)_i)$ instead.
%DIF >  If two nodes are equal in all comparisons, if the user provides sample times, the earlier sample time is given higher priority; otherwise, ties are broken arbitrarily.
%DIF >  %Given a rooted tree $T$, let $\D(u)$ denote the set of all leaf descendants of an internal node $u$ and 
%DIF >  %let $t(l)$ denote the sample time of leaf $l$.
%DIF >  %Let $\ta(u)=\sfrac{1}{\lvert \D(u)\rvert}\sum_{v\in \D(u)}{t(u)}$ denote the average sample time of leaf descendants below an internal $u$.
%DIF >  %Let $\bar{p}(l)$ be the path from $l$ to the root excluding $l$, and for each leaf $l$ let $\tb(l)=\min_{u\in \bar{p}(l)} \ta(u)$ be  the minimum average sample time assigned to any of it ancestor nodes (not including itself).
%DIF >  %ProACT orders leaves in the descending order (i.e., from the most recent to the oldest) of the tuple $(\tb(l),t(l))$.
%DIF >  %Note that $\ta(u)$ values can be computed in $\Theta(n)$ for all nodes $u$ in $T$ using a post-order traversal. 
%DIF >  %Because sorting is needed, for a tree with $n$ leaves,
%DIF >  %assuming branch lengths are fairly unique,
%DIF >  %the ProACT algorithm runs in $\bigO(n\log n)$ time. 
%DIF >  ProACT is very fast (scaling quasilinearly) and scalable methods exist  for the inferring~\supercite{Price2010,Nguyen2015} and rooting~\supercite{Mai2017} very large trees.

%To see the intuition behind ProACT, imagine that the phylogenetic tree is made up of several clades, such that \PLWH{s} belonging to each clade infect others with a fixed rate, but there is a different rates for different clades. 
%For each clade, the maximum likelihood estimate of the rate of that clade is the inverse of the average infection times below that node (assuming everyone in the clade is infected by others in the clade). 
%Then, ProACT is simply ordering clades based on their maximum likelihood rate, assuming they have a unique rate, and is ordering the clades based on these ML estimated rates. 
%Clearly, ordering based on rate of infection is justified.



\subsection{Simulations}
We used FAVITES to simulate a sexual contact network, transmission network, viral phylogeny, and viral sequences. 
\DIFdelbegin \DIFdel{FAVITES simulations have many parameters , and we chose a default set }\DIFdelend \DIFaddbegin \DIFadd{We chose  FAVITES parameters }\DIFaddend (Table~S2) \DIFdelbegin \DIFdel{aiming to emulate  10 years of }\DIFdelend \DIFaddbegin \DIFadd{to emulate  }\DIFaddend HIV transmission in San Diego (\DIFdelbegin \DIFdel{2005 -- 2014).
For the most part, we }\DIFdelend \DIFaddbegin \DIFadd{2005--2014).
We mostly }\DIFaddend used the parameters \DIFdelbegin \DIFdel{used in }\DIFdelend \DIFaddbegin \DIFadd{from }\DIFaddend \authorcite{Moshiri2018} \DIFdelbegin \DIFdel{,
}\DIFdelend \DIFaddbegin \DIFadd{with
}\DIFaddend %that sought to model the San Diego HIV epidemic from 2005 to 2014, 
\DIFdelbegin \DIFdel{but made modifications detailed blow }\DIFdelend \DIFaddbegin \DIFadd{some modifications }\DIFaddend to better capture reality.
We \DIFdelbegin \DIFdel{also carefully }\DIFdelend explored the impact of changing three of the the main parameters (Table~1) to obtain nine model conditions \DIFdelbegin \DIFdel{.
For each condition, we simulated }\DIFdelend \DIFaddbegin \DIFadd{(each replicated }\DIFaddend 20 \DIFdelbegin \DIFdel{replicates}\DIFdelend \DIFaddbegin \DIFadd{times)}\DIFaddend .
%\supercite{Moshiri2018}.



\begin{table*}[!t]
\begin{center}
\begin{tabular}{c c}
\toprule
\textbf{Parameter~~~~~~} & \textbf{Values}\\
\toprule
ART Initiation Rate ($\lambda_+$, year$^{-1}$)~~~~~~ & \textbf{1}, 2, 4\\
\hdashline
ART Termination Rate ($\lambda_-$, year$^{-1}$)~~~~~~ & 0.12 (0.25x), 0.24 (0.5x),\\
~~~~~~ & \textbf{0.48 (1x)}, 0.96 (2x), 1.92 (4x)\\
\hdashline
Expected Degree $\left(E_d\right)$~~~~~~ & \textbf{10}, 20, 30\\
\bottomrule
\end{tabular}
\end{center}
\caption{HIV simulation parameters. Values for the base model condition are shown in bold.}
\label{tab:favites}
\end{table*}


\paragraph{Sexual contact network.}
%In \authorcite{Moshiri2018}, sexual contact networks were simulated under the Barab\`asi--Albert (BA) model \supercite{Barabasi1999} such that the expected degree (i.e., average number of sexual contacts for a single individual) was 4.
To \DIFdelbegin \DIFdel{capture the }\DIFdelend \DIFaddbegin \DIFadd{simulate }\DIFaddend scale-free \DIFdelbegin \DIFdel{nature of the sexual contact network}\DIFdelend \DIFaddbegin \DIFadd{sexual contact networks}\DIFaddend , \authorcite{Moshiri2018} used the Barab\`asi--Albert (BA) model.\supercite{Barabasi1999}
\DIFdelbegin \DIFdel{In addition to the scale-free property, in }\DIFdelend HIV sexual networks \DIFdelbegin \DIFdel{, we typically observe many }\DIFdelend \DIFaddbegin \DIFadd{also typically include  }\DIFaddend densely-connected communities,\supercite{Rothenberg1998} a property the BA model fails to directly model.
To \DIFdelbegin \DIFdel{have control over the number of }\DIFdelend \DIFaddbegin \DIFadd{add }\DIFaddend communities,  we \DIFdelbegin \DIFdel{simulated }\DIFdelend \DIFaddbegin \DIFadd{independently simulated 20 BA }\DIFaddend sexual contact networks\DIFdelbegin \DIFdel{such that networks contained 20 BA communities}\DIFdelend ,
each with 5,000 individuals\DIFdelbegin \DIFdel{.
}\DIFdelend \DIFaddbegin \DIFadd{, and connected these communities randomly akin to the Erd\H os--R\'enyi model.}\supercite{Erdos1959}
\DIFaddend In the base condition, the expected degree of connection \DIFdelbegin \DIFdel{between an individual and somebody }\DIFdelend \textit{within} \DIFdelbegin \DIFdel{their }\DIFdelend \DIFaddbegin \DIFadd{the }\DIFaddend community was chosen to be 10,
and the expected degree \DIFdelbegin \DIFdel{between an individual and somebody }\DIFdelend \DIFaddbegin \DIFadd{of connection }\DIFaddend \textit{outside} \DIFdelbegin \DIFdel{their community was chosen to be }\DIFdelend \DIFaddbegin \DIFadd{the community was set to }\DIFaddend 1. 
\DIFdelbegin \DIFdel{Each community was simulated separately using the BA model and connections between communities were chosen uniformly at random, akin to the Erd\H os--R\'enyi model.}%DIFDELCMD < \supercite{Erdos1959}
%DIFDELCMD < %%%
\DIFdelend %DIF > Each community was simulated separately using the BA model and connections between communities were chosen uniformly at random, akin to the Erd\H os--R\'enyi model.\supercite{Erdos1959}
Estimates from the literature put the number of contacts at 3--4 during a single year.\supercite{Rosenberg2011}
Because our simulated sexual contacts remain static over the 10 year simulation period, we explore mean degrees $(E_d)$ between 10 and 30.


\paragraph{Transmission Model.}
Transmissions are modeled using a compartmental epidemiological model with 5 states (a simplified version of the \authorcite{Granich2009} model): Susceptible (S), Acute HIV Untreated (AU), Acute HIV Treated (AT), Chronic HIV Untreated (CU), and Chronic HIV Treated (CT). \DIFdelbegin \DIFdel{Individuals in state S (i.e., uninfected) }\DIFdelend \DIFaddbegin \DIFadd{Uninfected individuals (S) %DIF > (i.e., uninfected) 
}\DIFaddend can only transition to state AU. 
Each \DIFdelbegin \DIFdel{infected state $x\in\{\text{AU},\text{AT},\text{CU},\text{CT}\}$ defines a ``rate of infectiousness '' $\lambda_{\text{S},x}$: given an }\DIFdelend \DIFaddbegin \DIFadd{of infected states $\mathcal{I} = \{\text{AU},\text{AT},\text{CU},\text{CT}\}$
has its own ``infectiousness rate'' $\lambda_{\text{S},.}$. %DIF > given 
An }\DIFaddend uninfected individual $u$ \DIFdelbegin \DIFdel{in state S who has }\DIFdelend %DIF > in state S 
\DIFaddbegin \DIFadd{with }\DIFaddend $n_x$ sexual partners in state \DIFdelbegin \DIFdel{$x\in\{\text{AU},\text{AT},\text{CU},\text{CT}\}$, the transition of $u$ }\DIFdelend \DIFaddbegin \DIFadd{$x\in\mathcal{I}$ transitions }\DIFaddend from S to AU \DIFdelbegin \DIFdel{is }\DIFdelend \DIFaddbegin \DIFadd{by }\DIFaddend a Poisson process with rate \DIFdelbegin \DIFdel{$\lambda_u=\sum_{x\in\{\text{AU},\text{AT},\text{CU},\text{CT}\}}{n_{x}\lambda_{\text{S},x}}$. To mimic reality, where }\DIFdelend \DIFaddbegin \DIFadd{$\lambda_u=\sum_{x\in\mathcal{I}}{n_{x}\lambda_{\text{S},x}}$. Since }\DIFaddend ART significantly reduces the risk of transmission, rates are chosen such that $\lambda_{\text{S},\text{AU}} > \lambda_{\text{S},\text{CU}} > \lambda_{\text{S},\text{AT}} > \lambda_{\text{S},\text{CT}} \approx 0$. 
We refer to $\lambda_{AU,AT}=\lambda_{CU,CT}$ as the rate of ART initiation $(\lambda_+)$, and refer to $\lambda_{AT,AU}=\lambda_{CT,CU}$ as the rate of ART termination $(\lambda_-)$. 
We explore increasing or decreasing $\lambda_-$ by up to 4X, and explore increasing $\lambda_+$ by up to 4X (Table~1). 
\DIFdelbegin %DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend %\paragraph{Initialization.}
At the beginning of the  \DIFdelbegin \DIFdel{epidemic simulation, all initially uninfected individuals are placed in state S, and all initially }\DIFdelend \DIFaddbegin \DIFadd{simulation, %DIF > all initially uninfected individuals are placed in state S, and 
all initially }\DIFaddend infected (i.e., ``seed'') individuals are distributed among the 4 infected states according to their steady-state proportions \DIFdelbegin \DIFdel{(in contrast to
}\DIFdelend %DIF > (in contrast to
%The transmission model consists of four infected states (in descending order of infectiousness):
%Acute HIV without ART (AU), Chronic HIV without ART (CU), Acute HIV with ART (AT), and Chronic HIV with ART (CT).
\DIFaddbegin \DIFadd{(}\DIFaddend \authorcite{Moshiri2018} \DIFdelbegin \DIFdel{where initially infected individuals are all }\DIFdelend %DIF > where initially infected individuals are 
\DIFaddbegin \DIFadd{put all of them }\DIFaddend in state AU.)
%We randomly distribute initially infected individuals according to expected proportions of the states. 
\DIFdelbegin \DIFdel{To find }\DIFdelend \DIFaddbegin \DIFadd{The }\DIFaddend steady-state proportions \DIFdelbegin \DIFdel{, we ran simulations in which all seed individuals were in state AU, and we observed the proportion of individuals in each state over time, which reached a steady-state fairly early in the simulations }\DIFdelend \DIFaddbegin \DIFadd{were determined empirically }\DIFaddend (Fig.~\DIFdelbegin \DIFdel{S13}\DIFdelend \DIFaddbegin \DIFadd{S3}\DIFaddend ).


\paragraph{Viral \DIFaddbegin \DIFadd{and Sequence }\DIFaddend Evolution.}
\DIFdelbegin \DIFdel{Once the transmissions and sample times are obtained, the }\DIFdelend %DIF > Once the transmissions and sample times are obtained, 
\DIFaddbegin \DIFadd{The }\DIFaddend viral phylogeny evolves inside the transmission tree under a coalescent model  \DIFdelbegin \DIFdel{of evolution }\DIFdelend with logistic within-host viral population growth and a bottleneck event at the time of transmission (i.e., initial viral population size is 1).\supercite{Ratmann2017}
This process produces a separate viral phylogeny for each seed individual\DIFdelbegin \DIFdel{, so we also need a tree for seed individuals.
Each }\textit{\DIFdel{seed}} %DIFAUXCMD
\DIFdel{individual of the epidemic is the root of an independent viral phylogeny, and these }\DIFdelend \DIFaddbegin \DIFadd{.
%DIF > so we also need a tree for seed individuals. Each \textit{seed} individual of the epidemic is the root of an independent viral phylogeny, and 
These }\DIFaddend trees were merged by simulating a seed tree with one leaf per seed node under a non-homogeneous Yule model \supercite{LeGat2016} with rate function $\lambda(t)=e^{-t^2}+1$ scaled to have a height of 25 years to match  \DIFdelbegin \DIFdel{the estimate of the time of the most recent common ancestor of HIV in San Diego }\DIFdelend \DIFaddbegin \DIFadd{San Diego estimates }\DIFaddend (see \authorcite{Moshiri2018} for details). 
\DIFdelbegin %DIFDELCMD < 

%DIFDELCMD < %%%
\paragraph{\DIFdel{Sequence evolution.}}
%DIFAUXCMD
\addtocounter{paragraph}{-1}%DIFAUXCMD
\DIFdelend %DIF > \paragraph{Sequence evolution.}
Once the true tree was available, a mutation rate was sampled for each branch independently from a truncated normal random variable from 0 to infinity with a location parameter of 0.0008 and a scale parameter of 0.0005 to scale branch lengths from years to expected number of per-site mutations.\supercite{Moshiri2018}
Sequences were evolved according to the GTR model~\supercite{gtr}\DIFdelbegin \DIFdel{and down the trees with branch length in the mutation units.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend \DIFaddbegin \DIFadd{.
%DIF >  \authorcite{Moshiri2018} sampled all infected individuals exactly at the end time of the 10-year simulation period.
%DIF >  In reality, however,
%DIF >  HIV patients are typically sequenced when they first visit a clinic to receive ART.
%DIF >  Thus, it is expected that the terminal branch lengths of trees simulated in \authorcite{Moshiri2018} are artificially longer than would be expected.
Unlike  }\DIFaddend \authorcite{Moshiri2018} \DIFdelbegin \DIFdel{simulated sequencing of viral samples obtained from }\DIFdelend \DIFaddbegin \DIFadd{who sampled }\DIFaddend all infected individuals \DIFdelbegin \DIFdel{exactly at the end time of the 10-year simulation period.
In reality,
however,
HIV patients are typically sequenced when they first visit a clinic to receive ART.
Thus, it is expected that the terminal branch lengths of trees simulated in }%DIFDELCMD < \authorcite{Moshiri2018} %%%
\DIFdel{are artificially longer than would be expected.
Instead,
}\DIFdelend \DIFaddbegin \DIFadd{in year 10 of simulations,
}\DIFaddend we sample viral sequences \DIFdelbegin \DIFdel{from individuals }\DIFdelend the first time \DIFdelbegin \DIFdel{they begin }\DIFdelend \DIFaddbegin \DIFadd{an individual begins }\DIFaddend ART (i.e., \DIFdelbegin \DIFdel{the first time they enter }\DIFdelend state AT or CT).
\DIFdelbegin \DIFdel{Our current simulation better captures }\DIFdelend \DIFaddbegin \DIFadd{The goal is to model }\DIFaddend standards of care in the health care systems  \DIFdelbegin \DIFdel{of places }\DIFdelend that %do
afford wide-spread sequencing. 

\paragraph{Tree inference\DIFdelbegin \DIFdel{and subsampling}\DIFdelend . }
\DIFdelbegin \DIFdel{The simulations produce sequence data for all infected individual. 
}\DIFdelend %DIF > The simulations produce sequence data for all infected individual. 
Instead of providing ProACT with the simulated tree, 
using FastTree-2,\supercite{Price2010}
a phylogenetic tree was \textit{inferred} under the GTR+$\Gamma$  model from the sequences obtained in the first 9 years of the simulation, leaving the the 10th year for the evaluation step.
These trees were then MinVar-rooted\DIFdelbegin \DIFdel{using FastRoot}\DIFdelend .\supercite{Mai2017}
To measure the impact of imperfect sampling, we randomly subsampled the sequence datasets (at 25\%, 50\%, or 75\% levels)  before inferring phylogenies.
\DIFdelbegin \DIFdel{We ran ProACT on all trees (full and subsampled).
}\DIFdelend %DIF > We ran ProACT on all trees (full and subsampled).
% \subsubsection{Simulated data analysis}
% For each simulated sequence dataset,
% using FastTree-2 \supercite{Price2010},
% a phylogenetic tree was inferred under the GTR+$\Gamma$  model from the sequences obtained in the first 9 years of the simulation.
% These trees were then MinVar-rooted using FastRoot \supercite{Mai2017},
% and ProACT was run on the resulting trees.

\subsection{PIRC San Diego Dataset}
To test ProACT on real data, we used \DIFdelbegin \DIFdel{a Multiple Sequence Alignment (MSA) }\DIFdelend \DIFaddbegin \DIFadd{an alignment }\DIFaddend of 926 HIV-1 subtype B \textit{pol} sequences  \DIFdelbegin \DIFdel{from San Diego }\DIFdelend collected by the UC San Diego Primary Infection Resource Consortium (PIRC) between between 1996 and \DIFdelbegin \DIFdel{2018.
}\DIFdelend \DIFaddbegin \DIFadd{2018 in San Diego.
}\DIFaddend PIRC  is one of the largest longitudinal cohorts of \PLWH{s} in the United States. \DIFdelbegin \DIFdel{By design, PIRC }\DIFdelend \DIFaddbegin \DIFadd{PIRC design }\DIFaddend strives to include acute infections (as much as 40\% of \DIFdelbegin \DIFdel{recruited individuals are during acute or early  stages of }\DIFdelend \DIFaddbegin \DIFadd{individuals are recruited during  early  }\DIFaddend infection).  
\DIFdelbegin \DIFdel{Access to the data was obtained through a proposal submitted to PIRC.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend A phylogenetic tree was inferred from the \DIFdelbegin \DIFdel{MSA }\DIFdelend \DIFaddbegin \DIFadd{alignment }\DIFaddend under the GTR+$\Gamma$ model using FastTree-2\DIFdelbegin \DIFdel{,}\DIFdelend \supercite{Price2010} and \DIFdelbegin \DIFdel{the resulting tree }\DIFdelend was MinVar-rooted\DIFdelbegin \DIFdel{using FastRoot}\DIFdelend .\supercite{Mai2017} For each decile, \DIFdelbegin \DIFdel{using TreeSwift,}%DIFDELCMD < \supercite{Moshiri2020} %%%
\DIFdel{the full }\DIFdelend \DIFaddbegin \DIFadd{the }\DIFaddend tree was pruned to only contain samples obtained up to the end of that decile \DIFdelbegin \DIFdel{. }\DIFdelend \DIFaddbegin \DIFadd{using TreeSwift.}\supercite{Moshiri2020} %DIF > ProACT was run on each of the resulting trees.

\subsection{\DIFadd{Evaluation Procedure}}

\DIFaddend ProACT was run on each of the \DIFdelbegin \DIFdel{resulting trees .
}%DIFDELCMD < 

%DIFDELCMD < %%%
\subsection{\DIFdel{Evaluation Procedure}}
%DIFAUXCMD
\addtocounter{subsection}{-1}%DIFAUXCMD
%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend \DIFaddbegin \DIFadd{estimated trees of simulated and real data.
}\DIFaddend The metrics we use for evaluation are different between simulated and real data. 

\subsubsection{Simulated data}

\paragraph{Transmission/Person.}
\DIFdelbegin \DIFdel{To measure the efficacy of a given ProACT selection,
because the }\DIFdelend \DIFaddbegin \DIFadd{Because the }\DIFaddend true transmission histories are known in simulation,
we simply average the number of infections caused by highly prioritized individuals in a one-year time frame \textit{after} the  prioritization (i.e., year 10)
to obtain a raw outcome measure.
Then, we measure the cumulative moving average (CMA) of the outcome measure by the top \PLWH{s}. 
The higher the CMA in a prioritization, the higher the number of future transmissions from these {top} individuals, and thus, the higher the effectiveness of the prioritization. 
\DIFdelbegin %DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend Let $A=\{1,\ldots,n\}$ denote the \DIFdelbegin \DIFdel{first, \ldots, }\DIFdelend %DIF > first, \ldots, 
$n$
\DIFdelbegin \DIFdel{-th sampled individual in the current time step (}\DIFdelend %DIF > -th 
\DIFaddbegin \DIFadd{sampled individual }\DIFaddend years 1--9 \DIFdelbegin \DIFdel{in our simulations). 
For each individual $i$, let }\DIFdelend \DIFaddbegin \DIFadd{of our simulations. 
%DIF > For each individual $i$, l
Let }\DIFaddend $c(i)$ denote the number of individuals directly infected by \DIFaddbegin \DIFadd{individual }\DIFaddend $i$ in \DIFdelbegin \DIFdel{the next time step (}\DIFdelend year 10 \DIFdelbegin \DIFdel{in our simulations). 
Given any set }\DIFdelend \DIFaddbegin \DIFadd{of the simulations. 
For any subset }\DIFaddend of individuals $s\subseteq A$, let $C(s)=\frac{1}{|s|}\sum_{i\in s}{c(i)}$ denote \DIFdelbegin \DIFdel{the }\DIFdelend \DIFaddbegin \DIFadd{their }\DIFaddend average $c(i)$\DIFdelbegin \DIFdel{for all individuals $i\in s$.
Let $x=(x_1,\ldots,x_n)$ denote an ordering }\DIFdelend \DIFaddbegin \DIFadd{.
For an ordering $(x_1,\ldots,x_n)$  }\DIFaddend of $A$\DIFdelbegin \DIFdel{. The }\DIFdelend \DIFaddbegin \DIFadd{, the }\DIFaddend (unadjusted) CMA \DIFdelbegin \DIFdel{of $x$ }\DIFdelend up to $i$ is $C\left(\{x_1,\ldots,x_i\}\right)$.



\paragraph{Adjusted Transmissions/Person\DIFaddbegin \DIFadd{.}\DIFaddend } 
Sorting individuals by their transmissions in year 10  enables us to compute the optimal CMA curve, and the mean number of transmissions gives us the expected value of the CMA for a random prioritization.
Across experimental conditions, the maximum and random expectations vary.
Thus, to enable proper comparison of {effects of prioritization} across conditions,
we also report an adjusted CMA normalizing above the random prioritization and over the optimal prioritization.
Let \DIFdelbegin \DIFdel{$o=(o_1,\ldots,o_n)$ denote the ordering of $A$ in which elements are sorted in descending order of $c(i)$ }\DIFdelend \DIFaddbegin \DIFadd{$(o_1,\ldots,o_n)$ denote the optimal ordering }\DIFaddend (i.e., \DIFdelbegin \DIFdel{the optimal ordering}\DIFdelend \DIFaddbegin \DIFadd{in descending order of $c(i)$}\DIFaddend ), with ties broken arbitrarily. 
We \DIFdelbegin \DIFdel{defined }\DIFdelend \DIFaddbegin \DIFadd{define }\DIFaddend the adjusted CMA \DIFdelbegin \DIFdel{of $x$ }\DIFdelend up to $i$ as
\begin{equation*}%\label{eq:cma}
    \frac{C\left(\{x_1,\ldots,x_i\}\right)-C(A)}{C\left(\{o_1,\ldots,o_i\}\right)-C(A)}\; .
\end{equation*}
We use this equation to measure the effectiveness of a selection of the top $i$ individuals from each ordering of all individuals (exploring $i$ for 1 to 10\% of all  samples; i.e., $\frac{|A|}{10}$). 
For this  metric, 1 indicates the optimal ordering and 0 indicates an ordering that is no better than random (a negative value indicates an ordering that is \textit{worse} than random). 


\paragraph{Kendall's tau.} We also use Kendall's tau-b coefficient to measure the correlation between the optimal ordering and the ordering obtained using each method. 
Kendall's tau-b is a rank correlation coefficient adjusted for ties with values ranging between -1 and 1, with -1 signifying perfect inversion, 1 signifying perfect agreement, and 0 signifying the absence of association.\supercite{Kendall1938}


\subsubsection{Real data}


 


The sequences were sorted in ascending order of sample time and were divided into deciles.
For each decile,
two sets are formed: \textit{past} (sequences up to the decile) and \textit{future} (sequences after the decile). A  \DIFdelbegin \DIFdel{phylogenetic }\DIFdelend tree was inferred from the sequences in \textit{pre} \DIFdelbegin \DIFdel{under the GTR+$\Gamma$ model using FastTree-2 }%DIFDELCMD < \supercite{Price2010} %%%
\DIFdel{and MinVar-rooted}\DIFdelend \DIFaddbegin \DIFadd{as described earlier and was used as input to ProACT}\DIFaddend .
\DIFdelbegin %DIFDELCMD < \supercite{Mai2017}
%DIFDELCMD < %%%
\DIFdel{Using the resulting tree, ProACTordered the samples.
}\DIFdelend We then evaluated how our  outcome measure correlates with the position of each individual in the ordering. 
We quantify the correlation using Kendall's tau-b.

To define outcome measures, we cannot rely on \DIFdelbegin \DIFdel{known }\DIFdelend infections by individuals. Instead, we use the approach used by \authorcite{Wertheim2018}. 
We compute pairwise distances  between each sequence in \textit{pre} and each sequence in \textit{post} under the Tamura-Nei 93 (TN93) model \supercite{Tamura1993} 
\DIFdelbegin \DIFdel{using the }\texttt{\DIFdel{tn93}} %DIFAUXCMD
\DIFdel{tool of }\DIFdelend %DIF > using the \texttt{tn93} tool of
\DIFaddbegin \DIFadd{as in }\DIFaddend HIV-TRACE.\supercite{Pond2018}
A natural function to compute the risk of a given individual $u$ in \textit{pre},
similar to that proposed by \authorcite{Wertheim2018},
is to simply count the number of individuals in \textit{post} who are genetic links to $u$, i.e., $\sum_{v\in post}{I\left(d(u,v)\le1.5\%\right)}$.
In other words,
the score function is simply a step function with value 1 for all distances less than or equal to 1.5\% and 0 for all other distances.
However,
the selection of 1.5\% as the distance threshold,
despite being common practice in many HIV transmission clustering analyses,
is somewhat arbitrary,
and a step function exactly at this threshold may be overly strict
(e.g. should a pairwise distance of 1.51\% be ignored?).

To generalize this notion of scoring links,
we utilized three analytical score functions\DIFdelbegin \DIFdel{.
The first is the 
aforementioned step function$f_1(d)=I\left(d\le1.5\%\right)$.
The second is }\DIFdelend \DIFaddbegin \DIFadd{:
the 
%DIF > The first is the aforementioned 
step function, 
%DIF > $f_1(d)=I\left(d\le1.5\%\right)$.
%DIF > The second is 
}\DIFaddend a sigmoid function \DIFdelbegin \DIFdel{$f_2(d)=\frac{\lambda+1}{\lambda^{d/0.15}+\lambda}$ }\DIFdelend %DIF > $f_2(d)=\frac{\lambda+1}{\lambda^{d/0.15}+\lambda}$ 
with the choice of $\lambda=100$  and $\lambda=5$ (\DIFdelbegin \DIFdel{Fig.~S11) .
}\DIFdelend \DIFaddbegin \DIFadd{smoother versions of the step function) and
}\DIFaddend %a logarithmic function $f_4(d)=\frac{\ln(d+0.25)}{\ln(0.25)}$,
\DIFdelbegin \DIFdel{The third is an empirical scoring }\DIFdelend %DIF > The third is 
\DIFaddbegin \DIFadd{an empirical smooth step }\DIFaddend function learnt from the data by fitting a mixture model \DIFdelbegin \DIFdel{of three Gaussian random variables }\DIFdelend onto the distribution of pairwise TN93 distances 
\DIFdelbegin \DIFdel{$f_3(d)=\frac{p_1(x)}{p_1(x)+p_2(x)+p_3(x)}$, where $p_1(x)$ is the Probability Density Function (PDF) of the Gaussian component with smallest mean and $p_2(x)$ and $p_3(x)$ are the remaining Gaussian components
(}\DIFdelend %DIF > $f_3(d)=\frac{p_1(x)}{p_1(x)+p_2(x)+p_3(x)}$, where $p_1(x)$ is the density function of the Gaussian component with smallest mean and $p_2(x)$ and $p_3(x)$ are the remaining Gaussian components
\DIFaddbegin \DIFadd{(}\DIFaddend Fig.~\DIFdelbegin \DIFdel{S11).
Specifically, the three Gaussian fits were parameterized by ($\mu_1$=0.0191, $\sigma_1$=0.0103), ($\mu_2$=0.0609, $\sigma_2$=0.0118), and ($\mu_3$=0.118, $\sigma_3$=0.0468), respectively.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdel{For each of these }\DIFdelend \DIFaddbegin \DIFadd{S4).
%DIF >  TODOSUP
%DIF > Specifically, the three Gaussian fits were parameterized by ($\mu_1$=0.0191, $\sigma_1$=0.0103), ($\mu_2$=0.0609, $\sigma_2$=0.0118), and ($\mu_3$=0.118, $\sigma_3$=0.0468), respectively.
For each }\DIFaddend function,
for each decile\DIFdelbegin \DIFdel{to define }\textit{\DIFdel{pre}} %DIFAUXCMD
\DIFdel{and }\textit{\DIFdel{post}}%DIFAUXCMD
\DIFdel{, }\DIFdelend \DIFaddbegin \DIFadd{, %DIF > to define \textit{pre} and \textit{post},
}\DIFaddend we performed a Kendall's tau-b test to compare the prioritization approaches.\supercite{Kendall1938}
To generate a null distribution for \DIFdelbegin \DIFdel{visualization}\DIFdelend \DIFaddbegin \textit{\DIFadd{visualization}}\DIFaddend , we randomly shuffled the individuals in \textit{pre}\DIFdelbegin \DIFdel{repeatedly; note howeverthat the }\DIFdelend \DIFaddbegin \DIFadd{; however, }\DIFaddend \textit{p}-values \DIFdelbegin \DIFdel{that we report are }\textit{\DIFdel{p}}%DIFAUXCMD
\DIFdel{-values computed by }\DIFdelend \DIFaddbegin \DIFadd{are computed using }\DIFaddend the tau-b test \DIFdelbegin \DIFdel{, not empirically estimated from our repeated shuffling}\DIFdelend \DIFaddbegin \DIFadd{(not the empirical shuffling)}\DIFaddend .

%Recall that $t(l)$ denotes the sample time of sequence $l$. 
%For each sequence $l$ in \textit{pre}, let $N(l)$ denote the set of sequences in \textit{post} with a TN93 distance of at most 1.5\% from $l$.
%To estimate the rate of arrival of new links, we use $R(l)=\big(\sfrac{1}{\lvert N(l)\rvert}{\sum_{v\in N(l)}{t(v)-t(l)}}\big)^{-1}$ or $R(l)=0$ when no new link is established.
%Note that the inverse of mean arrival times is the maximum likelihood estimate of the rate if we assume the new links are added in a memoryless process (i.e, waiting times are exponentially distributed) and values are uncensored. 
%Thus, $R(l)=0$ implies no links in \textit{post}, low $R(l)$ implies a long time delay between $l$ and its \textit{post} links, and high $R(l)$ implies a short time delay between $l$ and its \textit{post} links.

%Given an ordering $x=\left(x_1,\ldots,x_n\right)$, let $R(x)=\left(R(x_1),\ldots,R(x_n)\right)$. 
%Ideally, $R(x)$ should be in the descending order and if the order is random $R(x)$ values will be randomly ordered.
%To test if an order is better than random, let $A(x)=\sum_{i=1}^{n} \sfrac{1}{i}\sum_{j=1}^{i}{R(x_j)}$ denote the area under the curve of the cumulative moving average (CMA) of $R(x)$.
%$A(x)=\sum C(x)$. 
%Larger values of $A(x)$ imply that individuals with high $R(x_i)$ are early in the ordering. The minimum and maximum possible of $A(x)$ among all orderings, $A_{\min}(x)$ and $A_{\max}(x)$, respectively, can be computed by sorting $R(x)$ in ascending and descending order, respectively, and computing the area under the CMA curve of each. From these, a test statistic between 0 and 1 can be obtained: 
%{$S(x)=\sfrac{\left(A(x)-A_{\min}(x)\right)}{\left(A_{\max}(x)-A_{\min}(x)\right)}$}. 
%To compute a $p$-value, the background distribution of $S$ is sampled via a permutation test by shuffling $x$ 1,000 times and computing $S$ from each shuffling.

\section{Results}
\DIFdelbegin \DIFdel{We evaluate ProACT on simulated and real data. 
}\DIFdelend %DIF > We evaluate ProACT on simulated and real data. 


\subsection{Simulation Results}

%In order to test ProACT's efficacy, we performed a series of simulation experiments in which we used FAVITES \supercite{Moshiri2018} to generate a sexual contact network, transmission network, viral phylogeny, and viral sequences emulating HIV transmission in San Diego from 2005 to 2014 (Material and Methods).




% \begin{figure}[!tp]
% \centering
% \includegraphics[width=0.365\textwidth]{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_ARTRATE-1.pdf}%
% \includegraphics[trim=55 00 00 00,clip,width=0.317\textwidth]{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_ARTRATE-2.pdf}%
% \includegraphics[trim=55 00 00 00,clip,width=0.317\textwidth]{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_ARTRATE-4.pdf}\\\vspace{-10pt}
% {\small (a)}\\
% \includegraphics[width=0.365\textwidth]{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_STOPRATE-1x.pdf}%
% \includegraphics[trim=55 00 00 00,clip,width=0.317\textwidth]{{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_STOPRATE-2x}.pdf}%
% \includegraphics[trim=55 00 00 00,clip,width=0.317\textwidth]{{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_STOPRATE-4x}.pdf}\\\vspace{-10pt}
% {\small (b)}\\
% \includegraphics[width=0.365\textwidth]{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_STOPRATE-1x.pdf}%
% \includegraphics[trim=55 00 00 00,clip,width=0.317\textwidth]{{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_STOPRATE-0.5x}.pdf}%
% \includegraphics[trim=55 00 00 00,clip,width=0.317\textwidth]{{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_STOPRATE-0.25x}.pdf}\\\vspace{-10pt}
% {\small (c)}\\
% \includegraphics[width=0.365\textwidth]{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_EXPDEGREE-10.pdf}%
% \includegraphics[trim=55 00 00 00,clip,width=0.317\textwidth]{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_EXPDEGREE-20.pdf}%
% \includegraphics[trim=55 00 00 00,clip,width=0.317\textwidth]{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_EXPDEGREE-30.pdf}\\\vspace{-10pt}
% {\small (d)}\\
% \caption{ProACT performance on datasets simulated using FAVITES.
% Cumulative Moving Average (CMA) of adjusted number of transmissions per person across the first decile of prioritized \PLWH for each simulation parameter set.
% The horizontal axis depicts the quantile of highest-prioritized \PLWH (e.g. $x=0.01$ denotes the top percentile),
% and the vertical axis depicts their adjusted average number of transmissions per person (1 indicates the optimal ordering, and 0 indicates an ordering that is no better than random).
% In our simulations, we varied three parameters of interest:
% (a) the rate of ART initiation $(\lambda_+)$, (b-c) the rate of ART termination $(\lambda_-)$, and (d) the expected degree of the sexual network $(E_d)$.
% The simulations were 10 years in length,
% prioritization was performed 9 years into the simulation, and the adjusted average number of transmissions per person was computed during the last year of the simulation.
% The curves labeled ``Cluster Growth'' denote prioritization by inferring transmission clusters using HIV-TRACE at year 9 of the simulation and sorting clusters in descending order of growth rate since year 8.
% The curves labeled with percentages denote subsampled datasets.
% All curves were calculated using 20 simulation replicates.
% }
% \label{fig:efficacy}
% \end{figure}


\begin{figure}[!tp]
\centering
%{\footnotesize
%~\hfill(a)\hfill (b)\hfill~\\
%\includegraphics[width=0.493\textwidth,trim={0 0 0 24pt},clip]{figs/results_efficacy_individual_SAMPLE-FIRSTART_STOPRATE-1x.pdf}
%\includegraphics[width=0.493\textwidth,trim={0 0 0 24pt},clip]{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_ARTRATE-1.pdf}
%\\
%~\hfill(c)\hfill (d)\hfill~\\
%\includegraphics[width=0.493\textwidth,trim={0 0 0 24pt},clip]{figs/results_efficacy_individual_n1000.pdf}
%\includegraphics[width=0.493\textwidth,trim={0 0 0 22pt},clip]{figs/results_efficacy_tau_individual.pdf}
\includegraphics[width=\textwidth]{figs/Fig2.pdf}
%}
\caption{Effectiveness of prioritization on simulated datasets. 
The simulations were 10 years in length,
prioritization was performed 9 years into the simulation, and the effectiveness of prioritization was computed during the last year of the simulation using four metrics (a-d).
``Cluster Growth'' denotes prioritization by inferring transmission clusters using HIV-TRACE at year 9 of the simulation and sorting clusters in descending order of growth rate since year 8.
All curves were calculated using 20 simulation replicates.
(a) Cumulative Moving Average (CMA) of the number of transmissions per person across the first decile of prioritized \PLWH{s} for the default simulation parameter set (see Fig.~\DIFdelbeginFL \DIFdelFL{S4 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{S8 }\DIFaddendFL for all model conditions, which show similar patterns).
The horizontal axis depicts the quantile of highest-prioritized \PLWH{s} (e.g. $x=0.01$ denotes the top percentile), and the vertical axis depicts their average number of transmissions per person.
Global average across all individuals (i.e., expectation under random ordering) is shown in red.
The curves labeled with percentages denote subsampled datasets.
(b) CMA of \textit{adjusted} number of transmissions per person for the default model condition (see Fig.~\DIFdelbeginFL \DIFdelFL{S5 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{S9 }\DIFaddendFL for all model conditions, which show similar patterns). 
For \textit{adjusted} Transmissions/Person, 1 indicates the optimal ordering and 0 indicates  random ordering.
All other settings are similar to part a. 
(c) Average of the raw number of transmissions per person for the top 1,000 individuals (see Fig.~\DIFdelbeginFL \DIFdelFL{S3 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{S5 }\DIFaddendFL for other counts) in a prioritized list vs. simulation parameter set (1,000 individuals correspond to 1\%--6\% of all individuals across conditions). The violin plots are across 20 replicates and contain box plots with medians shown as white dots.
Red horizontal lines show population mean (i.e., random prioritization).
%and distribution means shown as dashed grey lines.
(d) Kendall tau-b correlation between the optimal ordering of \PLWH{s} (i.e., based on their number of transmissions in year 10) and the orderings by the two prioritization methods.
See Figure~\DIFdelbeginFL \DIFdelFL{S6 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{S7 }\DIFaddendFL for subsampled data. 
Distributions are across 20 replicates and are shown for each simulation condition. 
}\label{fig:efficacy-main}
\end{figure}











%\subsubsection{ProACT outperforms random prioritization across conditions}
\subsubsection{Default condition}
ProACT dramatically increased the performance compared to random ordering according to all of our outcome measures (Fig.~2).
Focusing on the transmissions per person measure, while the population mean was 0.05, the ProACT's CMA was close to 0.15 for the top 1\% of prioritized \PLWH{s} and gradually reduced to 0.1 for the top 10\% (Fig.~2a).
The top 1,000 individuals in the ProACT ordering (3\% of the population)  transmitted {0.12} times (median across our 20 replicates), which was 2.4x  higher than the median population average (Fig.~2c; see also Fig.~\DIFdelbegin \DIFdel{S3 }\DIFdelend \DIFaddbegin \DIFadd{S5 }\DIFaddend for numbers other than 1,000).
As desired, selecting fewer people from the top of ProACT prioritization resulted in more transmissions per person (Fig.~2a).
Compared to optimal ordering, however, the adjusted score both increased and decreased as more individuals were selected (Fig.~2b). 
The adjusted metric shows that while ProACT substantially outperformed random ordering, it did not come close to the effectiveness that could be achieved using the (hypothetical) perfect ordering. 
The Kendall's tau-b \DIFdelbegin \DIFdel{correlation }\DIFdelend also showed a positive correlation between ProACT \DIFdelbegin \DIFdel{ordering }\DIFdelend and optimal ordering;  although the correlation coefficient is far from perfect (Fig.~2d), the correlations are statistically significant in all replicates ($p<10^{-9}$; see Fig.~\DIFdelbegin \DIFdel{S7a}\DIFdelend \DIFaddbegin \DIFadd{S6}\DIFaddend ).
%The most clear signal for benefits of prioritization (e.g, a high CMA) is obtained for up to 10\% top-priority individuals (though exact values depend on the model condition). 




%\subsubsection{ProACT outperforms cluster growth}
\authorcite{Wertheim2018} have presented a method for prioritizing \PLWH{s} by clustering individuals based on viral genetic distance, tracking the size of each cluster over time, and prioritizing clusters in descending order of the growth rate. The approach can be extended to also order individuals (i.e., individuals belonging to clusters with high growth rates are prioritized higher; see Methods for details).
ProACT consistently outperformed prioritization using cluster growth  (Figs.~2).
For example, the top 1,000 individuals according to cluster growth transmitted on average to 0.06 other people, which, while higher than the population average, was half the 0.12 transmissions per person according to ProACT. 
Kendall tau-b results similarly indicate that ProACT has better correlation with the optimal ordering.
% \TODO{
% The only exception was when the rate of stopping ART was lowered all the way to 0.25x, which corresponds to expected time of ART termination of 8.3 years.
% In this condition where adherence was at its highest,
% prioritization by cluster growth outperformed ProACT when using the full dataset.}

\subsubsection{Impact of simulation parameters}
We then tested the impact of three simulation parameters, namely the rate of stopping ART, the rate of starting ART, and the node degree in the sexual network (Figs.~2cd \DIFdelbegin \DIFdel{,~S4, and ~S5}\DIFdelend \DIFaddbegin \DIFadd{and S5--S7}\DIFaddend ).

As we increased the rate of stopping ART $\left(\lambda_{-}\right)$ (i.e., \DIFdelbegin \DIFdel{with }\DIFdelend lower adherence), the gap between ProACT and cluster growth grew.
For example, the mean number of transmissions per person among the top 1,000 individuals chosen using ProACT and cluster growth were respectively 0.169 and 0.076 (a 1.21x improvement) \DIFdelbegin \DIFdel{for the condition with }\DIFdelend \DIFaddbegin \DIFadd{when }\DIFaddend $\lambda_-=4$x (Fig.~2c). 
This 1.21x improvement briefly increased to 1.26x and subsequently decreased to 1.01x, 0.69x, and 0.63x improvement as we reduced the rate or ART termination to 2x, 1x, 0.5x, and 0.25x.
Kendall tau-b correlations show similar patterns (Fig.~2d);
while almost all replicates of $\lambda_-=4$x have $p<10^{-20}$, for the 0.25x case, all replicates have $p>10^{-10}$ and one of the replicates has $p>10^{-3}$  (Fig.~\DIFdelbegin \DIFdel{S7a}\DIFdelend \DIFaddbegin \DIFadd{S6}\DIFaddend ).

%Increasing the rate of infection, 
%As  $\lambda_{-}$ decreased, ProACT's performance compared to optimal ordering tended to decrease, whereas cluster growth's performance compared to optimal ordering tended to increase; however, ProACT continued to outperform cluster growth for all but the $\lambda_-=$0.25X condition (Fig.~S5b--c).


% \begin{figure}[!tp]
% \centering
% \includegraphics[width=0.493\textwidth]{figs/results_efficacy_individual_n100.pdf}
% \includegraphics[width=0.493\textwidth]{figs/results_efficacy_individual_n1000.pdf}\\
% \caption{Efficacy on datasets simulated using FAVITES. Average of the raw number of transmissions per person for the top $n$ individuals in a prioritized list vs. simulation parameter set across various values of $n$. The violin plots depicted are across 20 replicates and contain box plots with distribution medians shown as white dots and distribution means shown as dashed grey lines.}\label{fig:efficacy-vs-n}
% \end{figure}




As we increased the rate of starting ART $\left(\lambda_{+}\right)$ (i.e., with faster diagnoses), as expected, the raw number of new infections caused per capita  decreased (Fig.~2c, ~\DIFdelbegin \DIFdel{S4a}\DIFdelend \DIFaddbegin \DIFadd{S5}\DIFaddend ).
While ProACT remained effective in finding high priority individuals,
its performance compared to optimal ordering slightly degraded with higher $\lambda_{+}$ (Figs.~2d and ~\DIFdelbegin \DIFdel{S5a}\DIFdelend \DIFaddbegin \DIFadd{S7}\DIFaddend ).
Also, the gap between ProACT and cluster growth decreased slightly.
When observing the mean number of transmissions per person among the top 1,000 individuals chosen by each method (Fig.~2c),
ProACT gave a 1.01x, 1.03x, and 0.71x improvement over cluster growth for $\lambda_{+}$ set to 1x, 2x, and 4x, respectively.
%As $\lambda_{+}$ increased, neither ProACT nor prioritization by cluster growth were dramatically impacted, and ProACT consistently performed roughly twice as well as cluster growth (Fig.~S5a).


Changing the \DIFdelbegin \DIFdel{expected }\DIFdelend number of sexual contacts per person $\left(E_d\right)$, which controls the speed of spread, \DIFdelbegin \DIFdel{did not have uniform }\DIFdelend \DIFaddbegin \DIFadd{had nonuniform }\DIFaddend effects (Figs.~2cd). 
Increasing $E_d$ from 10 to 20 did not substantially impact the performance of ProACT. However, for $E_d=30$, we observed a small but noticeable reduction in the improvements of ProACT compared to the cluster growth (Figs.~2d and~\DIFdelbegin \DIFdel{S5d}\DIFdelend \DIFaddbegin \DIFadd{S7}\DIFaddend ).




\subsubsection{Impact of incomplete sampling}
Subsampling the total dataset to include $\sfrac{3}{4}$, $\sfrac{1}{2}$, or $\sfrac{1}{4}$ of all \PLWH{s} had only a marginal impact on the performance of ProACT according to the CMA metric  (Figs.~2ab,~\DIFdelbegin \DIFdel{S4,~S5}\DIFdelend \DIFaddbegin \DIFadd{S8,~S9}\DIFaddend ). 
%Inevitably, the raw number of new infections decreased as the dataset was subsampled (Fig.~S4). 
%The benefit of ProACT and cluster growth with respect to optimal ordering reduced slightly at the $\sfrac{1}{4}$ sampling level (Figs.~S5).
Only at 25\% sampling level did we observe a small reduction in the performance of ProACT compared to the optimal ordering. 
%Despite the general robustness, small degradation in accuracy are observed at the $\sfrac{1}{4}$ sampling level. 
For example, 
with $\lambda_{+}=2$x, ProACT's performance remained quite similar across $\geq \sfrac{1}{2}$ sampling levels, but a reduction in performance was observed for the $\sfrac{1}{4}$ sampling level for both ProACT and  cluster growth  (Fig.~\DIFdelbegin \DIFdel{S5a}\DIFdelend \DIFaddbegin \DIFadd{S7}\DIFaddend ).


According to Kendall's tau-b, which measures the entire order \DIFdelbegin \DIFdel{not }\DIFdelend \DIFaddbegin \DIFadd{rather than }\DIFaddend just the top individuals, there was a more noticeable degradation in performance due to sampling (Fig.~\DIFdelbegin \DIFdel{S6}\DIFdelend \DIFaddbegin \DIFadd{S7}\DIFaddend ). 
In particular, reduced sampling increased the \textit{variance} across replicate simulations (note the wider distributions for reduced sampling in Fig.~\DIFdelbegin \DIFdel{S6}\DIFdelend \DIFaddbegin \DIFadd{S7}\DIFaddend ).
Moreover, statistical significance of the correlations degrades with lower sampling (Fig.~\DIFdelbegin \DIFdel{S7c--e}\DIFdelend \DIFaddbegin \DIFadd{S6}\DIFaddend ).
With  $\sfrac{1}{4}$ sampling, unlike full sampling, many model conditions include {\em some} replicates where the ProACT ordering is not significantly better than random according to Kendall's tau-b.
%Interestingly, for  $\lambda_{-}<1$x, ProACT's performance on  $\sfrac{1}{4}$ sampled datasets \textit{improved} relative to more complete sampling.
%and interestingly, it actually outperformed all other levels of sampling for the most extreme condition. 
%However, the efficacy of prioritization by cluster growth remained fairly consistent  for  $\lambda_{-}<1$x (Fig.~S5b--c).
%Similarly, the performance of ProACT compared to optimal ordering improved with $\sfrac{1}{4}$ sampled datasets when sexual contact degree increased to $E_d\geq 20$ (Fig.~S5d).






% \begin{figure}[!tp]
% \centering
% {\footnotesize
% ~\hfill(a)\hfill (b)\hfill~\\
% \includegraphics[width=0.493\textwidth,trim={0 0 0 24pt},clip]{figs/results_efficacy_neighbors_SAMPLE-FIRSTART_STOPRATE-1x.pdf}
% \includegraphics[width=0.493\textwidth,trim={0 0 0 24pt},clip]{figs/results_efficacy_normalized_neighbors_SAMPLE-FIRSTART_ARTRATE-1.pdf}
% \\
% %\includegraphics[width=0.493\textwidth]{figs/results_efficacy_neighbors_n100.pdf}
% ~\hfill(c)\hfill (d)\hfill~\\
% \includegraphics[width=0.493\textwidth,trim={0 0 0 24pt},clip]{figs/results_efficacy_neighbors_n1000.pdf}
% \includegraphics[width=0.493\textwidth,trim={0 0 0 22pt},clip]{figs/results_efficacy_tau_neighbors.pdf}}
% \caption{Neighbor transmissions non-tau
% }\label{fig:efficacy-neighborss}
% \end{figure}




\DIFdelbegin %DIFDELCMD < \begin{figure}[!tp]
%DIFDELCMD < %%%
\DIFdelendFL \DIFaddbeginFL \begin{figure}[!t]
\DIFaddendFL \centering
%{\footnotesize
%~\hfill(a)\hfill (b)\hfill~\\
%\includegraphics[width=0.493\textwidth,trim={0 0 0 24pt},clip]{figs/results_efficacy_neighbors_SAMPLE-FIRSTART_STOPRATE-1x.pdf}
%\includegraphics[width=0.493\textwidth,trim={0 0 0 24pt},clip]{figs/results_efficacy_normalized_neighbors_SAMPLE-FIRSTART_ARTRATE-1.pdf}
%\\
%~\hfill(c)\hfill (d)\hfill~\\
%\includegraphics[width=0.47\textwidth,trim={0 0 0 22pt},clip]{figs/results_efficacy_neighbors_n1000.pdf}
%\includegraphics[width=0.5\textwidth,trim={0 0 0 22pt},clip]{figs/results_efficacy_tau_neighbors.pdf}%
%\\
%}
\includegraphics[width=\textwidth]{figs/Fig3.pdf}
\caption{Second order effects. 
(a) CMA of the number of infections from contacts of the top individuals according to each ordering; other settings similar to Fig.~2a. (b)
Similar to  part (a) but adjusted for random and optimal ordering.
(c)
Number of transmissions from neighbours for the top 1,000 individuals  in a prioritized list vs. simulation parameter set.
 (d) 
 Kendall tau-b correlation between the optimal ordering of \PLWH{s}  and their ordering by the two prioritization methods.
 See Figure~\DIFdelbeginFL \DIFdelFL{S8 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{S10 }\DIFaddendFL for subsampled data. 
 }
\label{fig:efficacy-neighbors}
\end{figure}




\subsubsection{Second order effects}
We next asked if prioritization is effective in detecting people whose contacts also transmit abundantly. 
To do so, we explored a new outcome measure: the total number of transmissions from all contacts of a \PLWH. 
Prioritizing \PLWH{s} whose contacts are likely to transmit can give public health officials a chance to find undiagnosed individuals (likely to transmit) through partner tracing from diagnosed individuals and to  prioritize PrEP for uninfected individuals in sexual contact with prioritized individuals. 
%For this outcome measure, like the previous one, we report both CMA of the  {transmissions per neighbour} and the Kendall Tau's-b coefficient with respect to the optimal ordering. 



\begin{figure}[!tp]
\centering
{\footnotesize
%\includegraphics[width=0.5\textwidth,trim={0 0 0 24pt},clip]{figs/results_efficacy_tau_contacts.pdf}%
\includegraphics[width=0.5\textwidth]{figs/Fig4.pdf}%
\\
}
\caption{ProACT captures sexual contact network degrees.
 Kendall tau-b correlation between the number of contacts of each individual and their ordering by the prioritization methods.
 See Figure~\DIFdelbeginFL \DIFdelFL{S10 }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{S12 }\DIFaddendFL for subsampled data. 
 }
\label{fig:efficacy-conc}
\end{figure}

Across all model parameters, ProACT ordering outperformed random ordering and cluster growth according to the number transmissions per neighbors (Fig.~3).
For example, contacts of the top 1,000 individuals according to ProACT transmitted to 2.23 individuals on average (median across replicates), which is more than twice the number of \DIFdelbegin \DIFdel{transmissions by contacts }\DIFdelend across all individuals in the network (1.08).
Just as with the previous outcome measure, advantages of ProACT over random prioritization or cluster growth were most pronounced for lower $\lambda_{+}$ and higher $\lambda_{-}$ (Fig.~3c). 
The Kendall tau-b coefficients for the correlation between ProACT and the optimal ordering were high (Figs.~3d and~\DIFdelbegin \DIFdel{S8}\DIFdelend \DIFaddbegin \DIFadd{S10}\DIFaddend ); in fact, they were \textit{higher} for the transmissions from contacts compared to transmissions from the prioritized person (e.g. median coefficient was 0.084 for contacts and 0.033 for the individuals in the default condition). 
These coefficients were highly significant across all models and sampling levels (Fig.~\DIFdelbegin \DIFdel{S9a}\DIFdelend \DIFaddbegin \DIFadd{S11}\DIFaddend ).
Thus, ProACT was even more effective in finding individuals with active contact than it was for finding individuals who were not suppressed. 
These results were largely robust to reduced sampling, showing similar patterns of average performance but increased variance across replicates (Fig.~\DIFdelbegin \DIFdel{S8 and S9c--e}\DIFdelend \DIFaddbegin \DIFadd{S10 and S11}\DIFaddend ).

Further interrogating the properties of an individual and their ordering, we observed a substantial correlation between the number of contacts of \PLWH{s} in the sexual network and their position in the ProACT ordering (Fig.~4). 
Thus, while ProACT only considers the phylogeny, it  prioritized those individuals that had high degrees in the sexual contact network (hidden to ProACT). 
These correlations were strongest for networks with high degree and weakest when the rate of diagnosis was very high. 
Reducing sampling did not substantially affect mean correlation but increased variance over replicates (Fig.~\DIFdelbegin \DIFdel{S10}\DIFdelend \DIFaddbegin \DIFadd{S12}\DIFaddend ).





\subsection{Real San Diego dataset}

%. Values range between -1 and 1, with -1 signifying perfect inversion, 1 signifying perfect agreement, and 0 signifying the absence of association.

On real datasets, unlike the simulated data, the desired outcome measure, the number of new transmissions per person, is not known. 
Instead, we have to use inferred relationships.
HIV-TRACE (used in our cluster growth approach) defines
a pair of \PLWH{s} as ``genetically linked''
if their sequences are very similar (TN93 distance below 1.5\%).
We similarly use the TN93 sequence similarity as an outcome measure, but in addition to using a fixed threshold, we also use smoother functions (Fig.~\DIFdelbegin \DIFdel{S11}\DIFdelend \DIFaddbegin \DIFadd{S13}\DIFaddend ).
%Following the HIV-TRACE approach, we call a pair of \PLWH{s} genetically linked if their sequences are very similar, as judged by the TN93 distance.
%We then use as our outcome measure the number of new links
We measure the number of linked individuals using a step function (1 if TN93 distance is below 1.5\% and 0 otherwise) 
%a more dispersed logarithmic function,
and an empirical smooth step function determined by fitting a mixture of three Gaussians to the distribution of pairwise TN93 distances (Material and Methods).
We also explore an analytical smooth step function (parameterized sigmoid).  
Note that, when the step function is used, our outcome measure (computed for future transmissions) is exactly the same as what the cluster growth method uses for prioritizing (albeit, using past data). 
Thus, it is reasonable to suspect that the step function may favor cluster growth. 
As we move to smoother functions of distance to count genetic links, our measure is expected to become less biased in favor of HIV-TRACE. 

\begin{figure}[t]
\centering
%\includegraphics[width=0.49\textwidth]{figs/results_tautest_norm-fit.pdf}%
%\includegraphics[width=0.49\textwidth]{figs/results_tautest_step.pdf}
%\includegraphics[width=0.49\textwidth]{figs/results_tautest_sigmoid-L100.pdf}%
%\includegraphics[width=0.49\textwidth]{figs/results_tautest_sigmoid-L5.pdf}
\includegraphics[width=\textwidth]{figs/Fig5.pdf}
\caption{Kendall's tau-b test results for ProACT ordering on real data using four  score functions: an empirical smooth step function, a strict step function around 1.5\%, and the sigmoid  score functions with $\lambda=100$ and $\lambda=5$. The full San Diego dataset was split into two sets (\textit{pre} and \textit{post}) at each decile (shown on the horizontal axis). The individuals in \textit{pre} were ordered using ProACT and by cluster growth, and they were given a  ``score'' computed using a  score function (see Methods). Kendall's tau-b correlation coefficient was computed for each ordering with respect to the optimal possible ordering (i.e., sorting in descending order of the  score). The null distribution was visualized by randomly shuffling the individuals in \textit{pre}, and test \textit{p}-values are shown in Table~2.}
\label{fig:tautest}\label{fig:tautest-sup}
\end{figure}



\begin{table*}[!thb]
\caption{Kendall's tau-b test for a null hypothesis that a given prioritization yields a total outcome measure no better than random. We show \textit{p}-values for the real San Diego dataset for the first through ninth deciles using two outcome measure functions. 
%These \textit{p}-values do not correct for multiple hypothesis testing. 
Tests that failed to reject the null hypothesis with (uncorrected) \textit{p}-value $< 0.00138$ (corresponding to $\alpha=0.05$ with a Bonferroni multiple hypothesis testing correct with $n=36$) are marked with \dag.\label{tab:tautest}}
\footnotesize
\centering
\hspace{-1.5cm}
{\tabcolsep=0pt\begin{tabular*}{\textwidth}{@{\extracolsep{\fill}}llllllllll@{}}\toprule
\multicolumn{10}{c}{Empirical Smooth Step Function (FastTree)}\\
& \multicolumn{1}{c}{10\%} & \multicolumn{1}{c}{20\%} & \multicolumn{1}{c}{30\%} & \multicolumn{1}{c}{40\%} & \multicolumn{1}{c}{50\%} & \multicolumn{1}{c}{60\%} & \multicolumn{1}{c}{70\%} & \multicolumn{1}{c}{80\%} & \multicolumn{1}{c}{90\%}\\
\midrule
GD+CG & $^\dag2\times10^{-3}$ & $^\dag2\times10^{-2}$ & $^{\ }5\times10^{-6}$ & $^{\ }2\times10^{-4}$ & $^{\ }5\times10^{-5}$ & $^{\ }6\times10^{-7}$ & $^{\ }2\times10^{-9}$ & $^{\ }2\times10^{-8}$ & $^{\ }2\times10^{-11}$\\
ProACT & $^{\ }5\times10^{-8}$ & $^{\ }1\times10^{-4}$ & $^{\ }6\times10^{-6}$ & $^{\ }2\times10^{-7}$ & $^{\ }2\times10^{-8}$ & $^{\ }2\times10^{-11}$ & $^{\ }1\times10^{-11}$ & $^{\ }1\times10^{-11}$ & $^{\ }1\times10^{-17}$\\
\midrule\\~\\[-1em]
\multicolumn{10}{c}{Step Function Around 1.5\%}\\
& \multicolumn{1}{c}{10\%} & \multicolumn{1}{c}{20\%} & \multicolumn{1}{c}{30\%} & \multicolumn{1}{c}{40\%} & \multicolumn{1}{c}{50\%} & \multicolumn{1}{c}{60\%} & \multicolumn{1}{c}{70\%} & \multicolumn{1}{c}{80\%} & \multicolumn{1}{c}{90\%}\\
\midrule
GD+CG & $^{\ }4\times10^{-12}$ & $^{\ }1\times10^{-19}$ & $^{\ }3\times10^{-28}$ & $^{\ }7\times10^{-25}$ & $^{\ }2\times10^{-19}$ & $^{\ }8\times10^{-12}$ & $^{\ }1\times10^{-17}$ & $^{\ }5\times10^{-14}$ & $^{\ }2\times10^{-25}$\\
ProACT & $^{\ }1\times10^{-5}$ & $^{\ }5\times10^{-8}$ & $^{\ }3\times10^{-7}$ & $^{\ }2\times10^{-10}$ & $^{\ }1\times10^{-6}$ & $^{\ }1\times10^{-6}$ & $^{\ }1\times10^{-4}$ & $^\dag7\times10^{-3}$ & $^{\ }4\times10^{-7}$\\
\bottomrule
\end{tabular*}
}{}
\end{table*}

Using \DIFdelbegin \DIFdel{both ProACT and }\DIFdelend \DIFaddbegin \DIFadd{ProACT or }\DIFaddend cluster growth to prioritize individuals results in orderings of individuals with positive Kendall's tau-b correlations to the number of future genetic links  regardless of the  \DIFdelbegin \DIFdel{time (i.e., decile ) }\DIFdelend \DIFaddbegin \DIFadd{decile }\DIFaddend and the function used to count genetic links (Fig.~5).  
These correlations are statistically significant in almost all cases  (Table~2). 
The  correlation coefficient  ranges between 0.4 (ProACT; 10\% time)  and 0.1 (cluster growth; 20\% time) for empirical function, and 
between 0.6 (cluster growth; 10\% time) and 0.1 (ProACT; 80\% time) for the step function.

The comparison between ProACT and cluster growth depends on the choice of the function to count links. 
When counting the number of links using the step function,
\DIFdelbegin \DIFdel{prioritization by }\DIFdelend cluster growth consistently outperforms ProACT for all deciles of the dataset.
These results should be interpreted with the caveat  that we count HIV-TRACE links both to prioritize and to evaluate accuracy. 
%, which is expected, as the Step Function is how HIV-TRACE defines genetic linkage when defining transmission clusters.
However, 
according to the empirical smooth step function learned from the TN93 distances, ProACT outperforms cluster growth in all except one time point, where they are tied.

To further test the impact of the link-counting function applied to TN93 distances on  the relative accuracy of methods, we used a sigmoid function to replace the step function while keeping the inflection point at 1.5\% (Fig.~\DIFdelbegin \DIFdel{S11}\DIFdelend \DIFaddbegin \DIFadd{S13}\DIFaddend ). 
We observed that as the outcome measure function becomes more smooth,
ProACT's performance improves with respect to prioritization by cluster growth (Fig.~5, Table~S1).
Based on the more smooth sigmoid function ($\lambda=5$), ProACT outperforms cluster growth in all but one case where they are tied.  
Thus, simply counting distances close to 1.5\% as partial links leads to evaluations that favor ProACT.



As time increases, both methods experience seemingly downward trends in their tau coefficients, but the null distribution of tau coefficients also tightens  (Fig.~5). Thus,  both methods consistently do significantly better than expected by random chance and there is no clear relationship between \textit{p}-values of individual tool and time (Table~2).
However, both for the step function and the sigmoid functions, ProACT's relative performance with respect to cluster growth tends to improved over time.

\section{Discussion}


We start by discussing observed results and then comment on practical implications of this paper both for public health and for future research. 


\subsection{Discussion of Results}

In our simulations, ProACT was the least effective in conditions with very low rate of ART termination, which correspond to very high adherence, or high rates of ART initiation.
As expected, the total number of new infections \DIFdelbegin \DIFdel{originated from }%DIFDELCMD < \PLWH{s} %%%
\DIFdelend \DIFaddbegin \DIFadd{per  }\PLWH \DIFaddend is low when adherence is high,  reducing the opportunity for improving the ordering.
Thus, ProACT is the most beneficial  \DIFdelbegin \DIFdel{in settings }\DIFdelend where termination of ART or late diagnosis lead to diagnosed individuals who transmit frequently. 
%This observation is consistent with the motivation we presented for the ProACT algorithm. 
%Recall that the motivation relied on identifying \PLWH{s} who have stopped being suppressed. 
%If most \PLWH{s} have been started on treatment and none ever stops treatment, prioritization loses its practical relevance, and relatedly, ProACT loses its statistical power.  
%We saw a similar effects when we increased the rate of ART $(\lambda_{+})$, which is  also not surprising as increasing $\lambda_{+}$ is in effect similar to reducing $\lambda_{-}$. 

ProACT was quite robust to impacts of subsampling individuals and only at $\sfrac{1}{4}$ sampling did we start to lose accuracy.
We remind the reader that a $\sfrac{1}{4}$ sampling does not mean that $\sfrac{1}{4}$ of all infected individuals are in the dataset. 
Rather, it means that $\sfrac{1}{4}$ of diagnosed individuals  are available to us. Recall that, in our model, diagnosed individuals are immediately  sequenced and put on ART (which they may or may not sustain). 
At any point in time, a large partition of individuals who are infected are not diagnosed and thus not sampled. 
In other words, the full sampling case should not be misunderstood as including undiagnosed people.
 Rather, lack of full sampling corresponds to a case where some \PLWH{s} are known to {\em some} clinic but are not included in the study, perhaps due to a lack of sequencing or data sharing. 

% These results have to be interpreted in the context of our adjusted metric, which measures benefits over random and below optimal ordering. 
% The per capita number of new infections from high-priority \PLWH{s} was  \textit{lowered} when we subsampled the datasets (Fig.~S4).
% Thus, as expected, when some \PLWH{s} are missing from the dataset available to a particular analysis, the overall effectiveness of identifying top priority \PLWH{s} reduces. 
% However, the effectiveness reduces equally for the optimal ordering and the ProACT method is not impacted any worse than optimal ordering is. 
% In fact, ProACT is in some cases impacted a bit less harshly than optimal ordering, hence the improvements in adjusted outcome with $\sfrac{1}{4}$ sampling. 
% One should also keep in mind that choosing $x$\% highest priority individuals from the full datasets results in 4x as many individuals as choosing the top $x$\% of the $\sfrac{1}{4}$ subsampled dataset.



% On the San Diego PIRC data, we observed reduced {\em p}-values as time progressed and the dataset became larger. Thus, the last decile (i.e., utilizing the first 90\% of the data to predict riskiness in the next 10\%) resulted in tight null distributions around 0, further highlight significance of . 
% %is most representative of prioritization efforts in reality: 
% In practical use, public health officials will want to provide a prioritization method all information they have collected thus far, and they will want to predict riskiness in the very near future.
% Thus, it can be argued that  90\%-training-10\%-testing  is the most relevant case to practice. 
% The results also suggest that the ProACT method works the best when several hundreds of \PLWH{s} are included in the analyses. 

ProACT far outperformed random ordering. However, we note that, despite the strong performance, there is much room left for future improvement: the {\em adjusted} outcome measure is consistently below 8\%  when selecting up to 10\% of top-priority \PLWH{s}.
Thus, there is much room for improvement in identifying high-value individuals.
It will be unrealistic to expect that any statistical method based solely on sequence data (and perhaps also commonly available metadata, such as sampling times) will be able to come close to the optimal ordering. 
Nevertheless, methods better than ProACT could likely be developed in the future. 
Moreover, here, we used ML methods to infer trees and used mutation rate branch lengths. 
We made these choices mostly for computational expediency. 
However, ProACT algorithm can be applied on the potentially more accurate Bayesian estimates of the phylogeny. 
Also, one can attempt to use ProACT after dating the tree. 
Whether either adjustment results in substantial improvements should be studied in the future. 

\subsection{Implications of Results}

We \DIFdelbegin \DIFdel{formalized }\DIFdelend \DIFaddbegin \DIFadd{proposed }\DIFaddend a useful approach for thinking about the effectiveness of \DIFdelbegin \DIFdel{public health intervention in }\DIFdelend molecular epidemiology. 
Instead of focusing on the accuracy of methods of reconstructing phylogenetic trees or transmission networks, a question fraught with difficulties, we asked a more practical question. 
Given molecular epidemiological data, can the methods, whether phylogenetic or clustering-based, prioritize \PLWH{s} for increased attention by public health?
Using molecular epidemiology for prioritization is, of course, not a new idea. For example, \authorcite{Wertheim2018} presented a method to prioritize \PLWH{s} based on the growth rates of their transmission clusters.
\authorcite{Vasylyeva2018} performed a phylogeographic analysis to reconstruct HIV movement among different locations in Ukraine in order to infer region-level risk prioritization.
Much earlier even, \authorcite{Mellors1996} predicted HIV patient prognosis by quantifying HIV RNA in plasma; predicted prognosis can subsequently be used as a prioritization rank.
However, we hope that our formal definition of the  \DIFdelbegin \DIFdel{problem as a computational question (i.e., prioritization), }\DIFdelend \DIFaddbegin \DIFadd{prioritization problem, }\DIFaddend in addition to our extensive simulations and developed metrics of evaluation, will stir further work in this area. 
\DIFdelbegin \DIFdel{As stated before, it seems }\DIFdelend \DIFaddbegin \DIFadd{It is }\DIFaddend likely that more advanced methods than our simple prioritization approach can improve performance beyond ProACT in the future. 



ProACT prioritizes individuals, not clusters.
Prioritizing \DIFdelbegin \DIFdel{treatment followup or partner tracing for }\DIFdelend %DIF > treatment followup or partner tracing for 
individuals based on their perceived risk of future transmission promises to be perhaps more effective than targeting clusters.
\DIFdelbegin \DIFdel{However, such targeted approaches also pose }\DIFdelend \DIFaddbegin \DIFadd{This targeted approach also poses }\DIFaddend ethical questions that 
have to be considered. 
\DIFdelbegin \DIFdel{For example, we }\DIFdelend \DIFaddbegin \DIFadd{We }\DIFaddend may not want the algorithm to be biased towards particular demographic attributes. 
ProACT does not use {\em any} metadata in its prioritization, reducing risks of such biases. 
\DIFdelbegin \DIFdel{It simply uses the viral phylogeny.
}\DIFdelend %DIF > It simply uses the viral phylogeny.
%, which, compared to other types of data, may lead to fewer biases. 
Nevertheless, it is possible that factors such as the depth of the sampling of a demographic group can in fact change branch length patterns in the phylogeny and make ProACT less or more effective for certain demographic groups. 
These broader implications of individual prioritization and impacts of demographics on the performance of ProACT should be studied more carefully in future. 

\DIFaddbegin \DIFadd{Our results indicated that ProACT ordering is a function of features of the sexual contact network. 
For example, we showed that ProACT orders correlate with the degree of nodes in the sexual network (e.g. Figure 4).
These results are significant given the fact that ProACT is given no direct information about the sexual network. 
The fact that ProACT captures contact network features means that, even if prioritized }\PLWH{s} \DIFadd{are already suppressed, their sexual contacts, identified using tracing, can be good targets for interventive care. 
}

\DIFaddend The main practical question  is what can be done with a prioritized list of known \PLWH{s}.
We mentioned that using followups, public health officials can try to ensure sustenance of ART for prioritized individuals, and using partner tracing, they can target PrEP and HIV testing to contacts of  prioritized individuals. Followups, PrEP, and targeted testing are all expensive and can benefit from prioritization. 
\DIFdelbegin \DIFdel{Interestingly, our results indicated that ProACT ordering is a function of features of the sexual contact}\DIFdelend \DIFaddbegin \DIFadd{Given a fixed budget, ProACT can help decide who should get %DIF > is getting
followups, who should be %DIF > is
traced, and who should be %DIF > is
tested frequently}\DIFaddend . 
For example, \DIFdelbegin \DIFdel{we showed that ProACT orders correlate with the degree of nodes in the sexual network.
These results are significant given the fact that ProACT is given no direct data the sexual network. 
The fact that ProACT captures (contact) network features means that even if a prioritized }%DIFDELCMD < \PLWH %%%
\DIFdel{is already on ART (and thus unlikely to transmit), his/her sexual contacts can be good targets for interventive care. 
}\DIFdelend \DIFaddbegin \DIFadd{assuming an epidemic is similar to our default simulation condition, if we afford to contact trace 1,000 known individuals and effectively prevent transmission from all
of their %DIF > those
contacts (using ART or PrEP), we would prevent 2,230 further transmissions using ProACT as opposed to only 1,080 if the 1,000 individuals are randomly chosen. 
Thus, ProACT is a way to target prevention strategies to increase their effectiveness. 
%DIF > If other  not a replacement for these strategies but 
}\DIFaddend 




One may \DIFdelbegin \DIFdel{wonder }\DIFdelend \DIFaddbegin \DIFadd{ask }\DIFaddend whether ordering by branch lengths \DIFdelbegin \DIFdel{will result }\DIFdelend \DIFaddbegin \DIFadd{results }\DIFaddend in orderings that fail to change with time to reflect changes in the epidemic.
To answer this question,
on the San Diego PIRC data, we asked how fast the  ProACT ordering changes as time progresses. To do so, we computed Kendall's tau-b correlations to the ProACT ordering obtained using only the first decile of the dataset (Fig.~\DIFdelbegin \DIFdel{S12}\DIFdelend \DIFaddbegin \DIFadd{S14}\DIFaddend ). There was a strong but diminishing correlation with the initial ordering. The correlations started at 1 (by definition) and gradually decreased in the ninth decile to 0.522. 
The results show that as desired,  ProACT orders do in fact change with time, albeit gradually.
The gradual change implies that certain individuals remain high-priority as time progresses. 
In practical use, ProACT ordering should be combined with clinical knowledge about the status of individual patients. For example, high priority individuals according to ProACT can be given lower priority if they manage to constantly remain suppressed with multiple followups.
\DIFaddbegin \DIFadd{Future versions of ProACT should allow users to provide a history of interventions and change the result accordingly. 
}\DIFaddend More broadly, the ProACT \DIFdelbegin \DIFdel{ordering }\DIFdelend should be considered one \DIFdelbegin \DIFdel{more tool }\DIFdelend \DIFaddbegin \DIFadd{of the tools }\DIFaddend for prioritizing clinical care, \DIFdelbegin \DIFdel{but other data , not incorporated into the algorithm, }\DIFdelend \DIFaddbegin \DIFadd{and other data that it does not incorporate }\DIFaddend should also be considered. 

Finally, a question faced by public health officials is whether the cost of targeting diagnosed individuals for followups and partner tracing is worth the potential reduction in future cases. 
The answer to that question will inevitably depend on who is targeted.
For example, in our default simulation case, targeting individuals randomly can directly prevent 0.05 transmissions per chosen person  in the next 12 months, whereas targeting top 1,000 individuals according to ProACT would directly target 0.12 transmissions. 
Thus, prioritization can in fact change the cost-benefit analyses. 
Moreover, given a prioritization, one can use simulations to predict the outcome measure for the top individuals (similar to Fig.~\DIFdelbegin \DIFdel{S5}\DIFdelend \DIFaddbegin \DIFadd{S9}\DIFaddend ) and use metrics such as quality-adjusted life-year (QALY) to estimate how many top individuals should be targeted for the cost to justify the benefits.

\subsubsection*{Code and data availability}
ProACT is freely available at \url{https://github.com/niemasd/ProACT} \DIFdelbegin \DIFdel{as an }\DIFdelend \DIFaddbegin \DIFadd{in }\DIFaddend open-source\DIFdelbegin \DIFdel{software project}\DIFdelend .
All experiment data are available online: \url{https://github.com/niemasd/ProACT-Paper-Final}.


\subsubsection*{Acknowledgements}
We thank Susan B. Little for providing the San Diego HIV sequence dataset used in this study \DIFdelbegin \DIFdel{.
}\DIFdelend \DIFaddbegin \DIFadd{(access obtained through a  PIRC proposal.)
}\DIFaddend We also thank Joel O. Wertheim and Sanjay R. Mehta for fruitful discussions that helped motivate the development of ProACT.



\bigskip\bigskip

\printbibliography
\DIFaddbegin 

\beginsupplement
%DIF > \clearpage


%DIF > % FIG S1: ALGORITHM
\begin{figure}[hp]
    \centering
    {\bf \DIFaddFL{ProACT Algorithm}}
    \begin{itemize}
        \item {\bf \DIFaddFL{Input:}} \DIFaddFL{A  }{\em \DIFaddFL{rooted}} \DIFaddFL{phylogenetic tree $T$ of viral samples. %DIF > and a list of sample times
        }\item {\bf \DIFaddFL{Define:}} 
        \begin{itemize}
            \item \DIFaddFL{$bl(u)$: the incident branch length of node $u$  in the rooted input tree $T$.
            Assume the incident branch length of the root of $T$ is 0.
            }\item \DIFaddFL{$a(u)$: a vector of ancestors of node $u$ (including $u$),
where $a(u)_1$ is $u$,
$a(u)_2$ is the parent of $u$,
$a(u)_3$ is the grandparent of $u$, etc.
}\item  \DIFaddFL{$r(u)$: denote the length of the path from node $u$ to the root of $T$,
i.e., $r(u)=\sum_{v\in a(u)}{bl(v)}$.
        }\end{itemize}
        \item 
{\bf \DIFaddFL{Output:}} 
\DIFaddFL{Sort the leaves of $T$ in ascending order of $bl(a(u)_1)$,
with ties broken by $bl(a(u)_2)$, then by $bl(a(u)_3)$, etc., and output the sorted list.
}\\
\DIFaddFL{Notes on sorting: 
}\begin{itemize}
    \item 
\DIFaddFL{For two leaves $u$ and $v$,
$|a(u)|$ may be less than $|a(v)|$,
in which case, for all $|a(u)|<i\le|a(v)|$,
$\frac{r(u)}{|a(u)|-1}$ (i.e., average branch length along the path from $u$ to the root of $T$) is compared with $bl(a(v)_i)$ instead.
}\item 
\DIFaddFL{If two nodes are equal in all comparisons, if the user provides sample times, the earlier sample time is given higher priority; otherwise, ties are broken arbitrarily.
}\end{itemize}
    \end{itemize}

%DIF > Given a rooted tree $T$, let $\D(u)$ denote the set of all leaf descendants of an internal node $u$ and 
%DIF > let $t(l)$ denote the sample time of leaf $l$.
%DIF > Let $\ta(u)=\sfrac{1}{\lvert \D(u)\rvert}\sum_{v\in \D(u)}{t(u)}$ denote the average sample time of leaf descendants below an internal $u$.
%DIF > Let $\bar{p}(l)$ be the path from $l$ to the root excluding $l$, and for each leaf $l$ let $\tb(l)=\min_{u\in \bar{p}(l)} \ta(u)$ be  the minimum average sample time assigned to any of it ancestor nodes (not including itself).
%DIF > ProACT orders leaves in the descending order (i.e., from the most recent to the oldest) of the tuple $(\tb(l),t(l))$.
%DIF > Note that $\ta(u)$ values can be computed in $\Theta(n)$ for all nodes $u$ in $T$ using a post-order traversal.   
\caption{
\DIFaddFL{The algorithm of ProACT. 
Because sorting is needed, for a tree with $n$ leaves,
assuming branch lengths are fairly unique,
the ProACT algorithm runs in $\bigO(n\log n)$ time. 
 }}
    \label{fig:my_label}
\end{figure}

\clearpage

%DIF > % FIG S2: AVG BL OVER TIME, TRUE TREES
\begin{figure}[!h]
\centering
\includegraphics[width=0.95\textwidth]{figs/avg_edgelength_vs_time_true.pdf}\\
\caption{{As time progresses, the true incident branch length of each individual tends to decrease. This holds in inferred phylogenies as well (Fig.~\ref{fig:diagram}d).}}
\label{fig:true-bl-vs-time}
\end{figure}

\clearpage

%DIF > % FIG S3: PROP IN EACH STATE OVER TIME (STEADY-STATE)
\begin{figure}[!h]
\centering
\includegraphics[width=0.95\textwidth]{figs/prop_state_vs_time.pdf}
\caption{Proportion of individuals in each infected state (AU, AT, CU, and CT) vs. time in simulations in which all seed individuals at time 0 were placed in state AU.
These results are used to determine the steady-state.
\DIFaddFL{
We ran simulations in which all seed individuals were in state AU, and we observed the proportion of individuals in each state over time, which reached a steady-state fairly early in the simulations, as can be seen in the figure above. }}
\label{fig:prop-state-vs-time}
\end{figure}

%DIF > % FIG S4: PAIRWISE TN93 DISTS OF REAL SD DATA
\begin{figure}[!h]
\centering
\includegraphics[width=0.95\textwidth]{figs/sd.tn93.real.pdf}
\caption{{Pairwise distances of real San Diego sequences, corrected under the Tamura-Nei 93 (TN93) model of sequence evolution.}}
\label{fig:sd-real-tn93}
\end{figure}

\clearpage

%DIF > % FIG S5: EFFECTIVENESS (TRANS/PERSON) FOR ALL N
\begin{figure}[!h]
\centering
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_individual_n100.pdf}
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_individual_n1000.pdf}\\
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_individual_n200.pdf}
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_individual_n2000.pdf}\\
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_individual_n500.pdf}
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_individual_n5000.pdf}\\
\caption{{Effectiveness on datasets simulated using FAVITES. Average of the raw number of transmissions per person for the top $n$ individuals in a prioritized list vs. simulation parameter set across various values of $n$. The violin plots depicted are across 20 replicates and contain box plots with distribution medians shown as white dots and distribution means shown as dashed grey lines.}}\label{fig:efficacy-vs-n}
\end{figure}

\clearpage

%DIF > % FIG S6: TAU-B EFFECTIVENESS (TRANS/PERSON) P-VALUES
\begin{figure}[!h]
\centering
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_tau_pvalue_individual.pdf}
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_tau_pvalue_individual_sub75.pdf}\\
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_tau_pvalue_individual_sub50.pdf}
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_tau_pvalue_individual_sub25.pdf}\\
\caption{{Statistical significance of Tau-b correlation of effectiveness measure (number of transmissions per person for the top $n=1000$ individuals) between prioritization tools and the (hypothetical) optimal ordering.}}\label{fig:tau-b-pvalues}
\end{figure}

\clearpage

%DIF > % FIG S7: TAU-B EFFECTIVENESS (TRANS/PERSON) VALUES
\begin{figure}[!h]
\centering
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_tau_individual.pdf}
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_tau_individual_sub75.pdf}\\
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_tau_individual_sub50.pdf}
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_tau_individual_sub25.pdf}\\
\caption{{Tau-b correlation coefficients of effectiveness measure (number of transmissions per person for the top $n=1000$ individuals) between prioritization tools and the (hypothetical) optimal ordering.}}\label{fig:tau-b-values}
\end{figure}

\clearpage

%DIF > % FIG S8: EFFECTIVENESS CMAS
\begin{figure}[!h]
\centering
\includegraphics[width=0.32\textwidth]{figs/results_efficacy_individual_SAMPLE-FIRSTART_ARTRATE-4.pdf}
\includegraphics[width=0.32\textwidth]{figs/results_efficacy_individual_SAMPLE-FIRSTART_ARTRATE-2.pdf}
\includegraphics[width=0.32\textwidth]{figs/results_efficacy_individual_SAMPLE-FIRSTART_ARTRATE-1.pdf}\\
\includegraphics[width=0.32\textwidth]{figs/results_efficacy_individual_SAMPLE-FIRSTART_STOPRATE-0.25x.pdf}
\includegraphics[width=0.32\textwidth]{figs/results_efficacy_individual_SAMPLE-FIRSTART_STOPRATE-0.5x.pdf}
\includegraphics[width=0.32\textwidth]{figs/results_efficacy_individual_SAMPLE-FIRSTART_STOPRATE-2x.pdf}\\
\includegraphics[width=0.32\textwidth]{figs/results_efficacy_individual_SAMPLE-FIRSTART_STOPRATE-4x.pdf}
\includegraphics[width=0.32\textwidth]{figs/results_efficacy_individual_SAMPLE-FIRSTART_EXPDEGREE-20.pdf}
\includegraphics[width=0.32\textwidth]{figs/results_efficacy_individual_SAMPLE-FIRSTART_EXPDEGREE-30.pdf}\\
\caption{{Cumulative Moving Average (CMA) of the number of transmissions per person across the first decile of prioritized samples for all model conditions.}}\label{fig:cma-trans-individual-all}
\end{figure}

\clearpage

%DIF > % FIG S9: ADJUSTED EFFECTIVENESS CMAS
\begin{figure}[!h]
\centering
\includegraphics[width=0.32\textwidth]{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_ARTRATE-4.pdf}
\includegraphics[width=0.32\textwidth]{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_ARTRATE-2.pdf}
\includegraphics[width=0.32\textwidth]{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_ARTRATE-1.pdf}\\
\includegraphics[width=0.32\textwidth]{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_STOPRATE-0.25x.pdf}
\includegraphics[width=0.32\textwidth]{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_STOPRATE-0.5x.pdf}
\includegraphics[width=0.32\textwidth]{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_STOPRATE-2x.pdf}\\
\includegraphics[width=0.32\textwidth]{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_STOPRATE-4x.pdf}
\includegraphics[width=0.32\textwidth]{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_EXPDEGREE-20.pdf}
\includegraphics[width=0.32\textwidth]{figs/results_efficacy_normalized_individual_SAMPLE-FIRSTART_EXPDEGREE-30.pdf}\\
\caption{{Cumulative Moving Average (CMA) of the adjusted number of transmissions per person across the first decile of prioritized samples for all model conditions.}}\label{fig:cma-adj-trans-individual-all}
\end{figure}

\clearpage

%DIF > % FIG S10: TAU-B EFFECTIVENESS (NEIGHBOR TRANS/PERSON) VALUES
\begin{figure}[!h]
\centering
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_tau_neighbors.pdf}
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_tau_neighbors_sub75.pdf}\\
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_tau_neighbors_sub50.pdf}
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_tau_neighbors_sub25.pdf}\\
\caption{{Tau-b correlation coefficients of effectiveness measure (number of neighbor transmissions per person for the top $n=1000$ individuals) between prioritization tools and the (hypothetical) optimal ordering.}}\label{fig:tau-b-values-neighbors}
\end{figure}

\clearpage

%DIF > % FIG S11: TAU-B EFFECTIVENESS (NEIGHBOR TRANS/PERSON) P-VALUES
\begin{figure}[!h]
\centering
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_tau_pvalue_neighbors.pdf}
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_tau_pvalue_neighbors_sub75.pdf}\\
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_tau_pvalue_neighbors_sub50.pdf}
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_tau_pvalue_neighbors_sub25.pdf}\\
\caption{{Statistical significance of Tau-b correlation of effectiveness measure (number of neighbor transmissions per person for the top $n=1000$ individuals) between prioritization tools and the (hypothetical) optimal ordering.}}\label{fig:tau-b-pvalues-neighbors}
\end{figure}

\clearpage

%DIF > % FIG S12: TAU-B EFFECTIVENESS (CONTACTS) VALUES
\begin{figure}[!h]
\centering
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_tau_contacts.pdf}
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_tau_contacts_sub75.pdf}\\
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_tau_contacts_sub50.pdf}
\includegraphics[width=0.475\textwidth]{figs/results_efficacy_tau_contacts_sub25.pdf}\\
\caption{{Tau-b correlation coefficients of effectiveness measure (number of contacts per person for the top $n=1000$ individuals) between prioritization tools and the (hypothetical) optimal ordering.}}\label{fig:tau-b-values-contacts}
\end{figure}

\clearpage

%DIF > % FIG S13: SCORE FUNCTIONS
\begin{figure}[!h]
\centering
\includegraphics[width=0.95\textwidth]{figs/riskiness_score_functions_zoomed.pdf}
%DIF > \includegraphics[width=0.495\textwidth]{figs/riskiness_score_functions.pdf}
\caption{Score functions vs. pairwise sequence distance.
\DIFaddFL{To generalize the notion of scoring links based on the step function used in HIV-Trace (orange line),
we utilized three analytical score functions shown here.
The step function is $f_1(d)=I\left(d\le1.5\%\right)$.
The second function is a sigmoid function $f_2(d)=\frac{\lambda+1}{\lambda^{d/0.15}+\lambda}$ with the choice of $\lambda=100$  and $\lambda=5$.
%DIF > a logarithmic function $f_4(d)=\frac{\ln(d+0.25)}{\ln(0.25)}$,
The third is an empirical scoring function learnt from the data by fitting a mixture model of three Gaussian random variables onto the distribution of pairwise TN93 distances $f_3(d)=\frac{p_1(x)}{p_1(x)+p_2(x)+p_3(x)}$, where $p_1(x)$ is the density function of the Gaussian component with smallest mean and $p_2(x)$ and $p_3(x)$ are the remaining Gaussian components.
Specifically, the three Gaussian fits were parameterized by ($\mu_1$=0.0191, $\sigma_1$=0.0103), ($\mu_2$=0.0609, $\sigma_2$=0.0118), and ($\mu_3$=0.118, $\sigma_3$=0.0468), respectively.
}}
\label{fig:scorefuncs}
\end{figure}

\clearpage

%DIF > % FIG S14: TAU-B PROACT WITH SELF REAL DATA
\begin{figure}[!h]
\centering
\includegraphics[width=0.95\textwidth]{figs/results_tautest_first-proact.pdf}
\caption{{Kendall's tau-b test results for ProACT ordering with respect to the ProACT ordering obtained with only the first decile of the dataset. The full San Diego dataset was split into two sets (}\textit{\DIFaddFL{pre}} \DIFaddFL{and }\textit{\DIFaddFL{post}}\DIFaddFL{) at each decile (shown on the horizontal axis). The individuals in }\textit{\DIFaddFL{pre}} \DIFaddFL{were ordered using ProACT and by cluster growth. Kendall's tau-b correlation coefficient was computed for each ordering with respect to the ProACT ordering at the first decile. The null distribution was visualized by randomly shuffling the individuals in }\textit{\DIFaddFL{pre}}\DIFaddFL{.}}
\label{fig:tautest-first-proact}
\end{figure}

\clearpage


%% TABLE S1: TAU-B SCORE FUNCTIONS REAL DATA
\begin{table}[!h]
\caption{Kendall's tau-b test for a null hypothesis that a given prioritization yields a total outcome measure no better than random. We show \textit{p}-values for the real San Diego dataset for the first through ninth deciles using two outcome measure functions. 
Tests that failed to reject the null hypothesis with (uncorrected) \textit{p}-value $< 0.00138$ (corresponding to $\alpha=0.05$ with a Bonferroni multiple hypothesis testing correct with $n=36$) are marked with \dag.\label{tab:tautest-sup}}
\footnotesize
\centering
\hspace{-1.5cm}
{\tabcolsep=0pt\begin{tabular*}{\textwidth}{@{\extracolsep{\fill}}llllllllll@{}}\toprule
\multicolumn{10}{c}{Sigmoid Function $(\lambda=5)$}\\
& \multicolumn{1}{c}{10\%} & \multicolumn{1}{c}{20\%} & \multicolumn{1}{c}{30\%} & \multicolumn{1}{c}{40\%} & \multicolumn{1}{c}{50\%} & \multicolumn{1}{c}{60\%} & \multicolumn{1}{c}{70\%} & \multicolumn{1}{c}{80\%} & \multicolumn{1}{c}{90\%}\\
\midrule
GD + Cluster Growth & $^{\ }6\times10^{-4}$ & $^\dag6\times10^{-3}$ & $^{\ }3\times10^{-7}$ & $^{\ }5\times10^{-5}$ & $^{\ }8\times10^{-6}$ & $^{\ }2\times10^{-7}$ & $^{\ }8\times10^{-8}$ & $^{\ }1\times10^{-6}$ & $^{\ }1\times10^{-10}$\\
ProACT (FastTree) & $^{\ }1\times10^{-8}$ & $^{\ }8\times10^{-5}$ & $^{\ }2\times10^{-6}$ & $^{\ }5\times10^{-8}$ & $^{\ }1\times10^{-8}$ & $^{\ }1\times10^{-11}$ & $^{\ }1\times10^{-10}$ & $^{\ }3\times10^{-11}$ & $^{\ }1\times10^{-17}$\\
\midrule\\~\\[-1em]
\multicolumn{10}{c}{Sigmoid Function $(\lambda=100)$}\\
& \multicolumn{1}{c}{10\%} & \multicolumn{1}{c}{20\%} & \multicolumn{1}{c}{30\%} & \multicolumn{1}{c}{40\%} & \multicolumn{1}{c}{50\%} & \multicolumn{1}{c}{60\%} & \multicolumn{1}{c}{70\%} & \multicolumn{1}{c}{80\%} & \multicolumn{1}{c}{90\%}\\
\midrule
GD + Cluster Growth & $^{\ }1\times10^{-8}$ & $^{\ }2\times10^{-11}$ & $^{\ }6\times10^{-20}$ & $^{\ }3\times10^{-24}$ & $^{\ }2\times10^{-23}$ & $^{\ }5\times10^{-17}$ & $^{\ }3\times10^{-15}$ & $^{\ }6\times10^{-11}$ & $^{\ }4\times10^{-16}$\\
ProACT (FastTree) & $^{\ }2\times10^{-10}$ & $^{\ }7\times10^{-9}$ & $^{\ }3\times10^{-11}$ & $^{\ }4\times10^{-18}$ & $^{\ }9\times10^{-17}$ & $^{\ }4\times10^{-20}$ & $^{\ }7\times10^{-15}$ & $^{\ }2\times10^{-12}$ & $^{\ }1\times10^{-20}$\\
% \midrule\\~\\[-1em]
% \multicolumn{10}{c}{End-Time Incident Branch Length (FastTree)}\\
% & \multicolumn{1}{c}{10\%} & \multicolumn{1}{c}{20\%} & \multicolumn{1}{c}{30\%} & \multicolumn{1}{c}{40\%} & \multicolumn{1}{c}{50\%} & \multicolumn{1}{c}{60\%} & \multicolumn{1}{c}{70\%} & \multicolumn{1}{c}{80\%} & \multicolumn{1}{c}{90\%}\\
% \midrule
% GD + Cluster Growth & $^{\ }7\times10^{-6}$ & $^{\ }2\times10^{-16}$ & $^{\ }3\times10^{-24}$ & $^{\ }5\times10^{-32}$ & $^{\ }4\times10^{-49}$ & $^{\ }2\times10^{-49}$ & $^{\ }6\times10^{-58}$ & $^{\ }4\times10^{-78}$ & $^{\ }2\times10^{-97}$\\
% ProACT (FastTree) & $^{\ }8\times10^{-15}$ & $^{\ }6\times10^{-39}$ & $^{\ }1\times10^{-70}$ & $^{\ }5\times10^{-103}$ & $^{\ }4\times10^{-143}$ & $^{\ }5\times10^{-175}$ & $^{\ }1\times10^{-227}$ & $^{\ }3\times10^{-277}$ & $^{\ }3\times10^{-308}$\\
\bottomrule
\end{tabular*}
}{}
\end{table}

%\clearpage

%% TABLE S2: FAVITES PARAMS
\begin{table}[!h]
\begin{center}
\begin{tabular}{c c}
\toprule
\textbf{Parameter} & \textbf{Default Value}\\
\toprule
Number of Contact Network Communities & 20\\
Number of Individuals per Community & 5,000\\
Mean Number of Edges Within Community & 10\\
Mean Number of Edges Outside Community & 1\\
\hdashline
Number of Seed Individuals & 15,000\\
Seed Selection Model & Uniformly Random\\
Seed State Frequencies $\{AU,AT,CU,CT\}$ & $\{0.0033,0.0006,0.3396,0.6565\}$\\
Expected Transition Time AU$\rightarrow$CU & 6 weeks\\
Expected Transition Time AT$\rightarrow$CT & 12 weeks\\
Expected ART Initiation Time & 1 year\\
Expected ART Termination Time & 25 months\\
Rates of Infectiousness $\{AU,AT,CU,CT\}$ & $\{0.1125,0.005625,0.0225,0.000\}$\\
\hdashline
Seed Sequence Phylogenetic Model & Non-Homogeneous Yule Process\\
Seed Phylogeny Height & 25 years\\
Seed Phylogeny Speciation Rate Function & $\exp(-t^2)+1$\\
Mutation Rate Model & Truncated Normal\\
Mutation Rate Location & 0.0008\\
Mutation Rate Scale & 0.0005\\
Mutation Rate Minimum & 0\\
Mutation Rate Maximum & $\infty$\\
\hdashline
Viral Sequence Type & HIV-1 Subtype B \textit{pol}\\
Sequence Evolution Model & GTR+$\Gamma$\\
GTR State Frequencies $\{p_A,p_C,p_G,p_T\}$ & $\{0.392,0.165,0.212,0.232\}$\\
GTR Transition Rates $\{\lambda_{AC},\lambda_{AG},\lambda_{AT},\lambda_{CG},\lambda_{CT},\lambda_{GT}\}$ & $\{1.766,9.588,0.692,0.863,10.283,1.000\}$\\
GTR Gamma Distribution Shape & 0.405\\
\hdashline
Viral Population Growth Rate Model & Logistic\\
Viral Population Growth Rate & 2.851904\\
Initial Viral Population Size & 1\\
Viral Population T50 & -2\\
Number of Sampled Lineages per Person & 1\\
Time of Sampling & ART Initiation\\
\bottomrule
\end{tabular}
\end{center}
\caption{Default FAVITES simulation parameters.}
\label{tab:config-file}
\end{table}

\end{document}
